---
title: "Genetics tools"
date: 2022-08-26T16:50:25+08:00
categories:
- category
- subcategory
tags:
- Gentics
- Mouse
- Drosophila
- Zebrafish
keywords:
- tech
comments:       false
showMeta:       false
showActions:    false
thumbnailImage: /img/refreshingyourstrains_blogpost.jpg
thumbnailImagePosition: left
#thumbnailImage: //example.com/image.jpg
---
> Progress in science depends on new techniques, new discoveries and new ideas, probably in that order.\
--- Sydney Brenner

<!--more-->

![sydneybrenner1](/img/quotes_sydney_brenner.jpg)

---
# Caenorhabditis elegans(C. elegans)

---
# Mus musculus
![mus129s1](/img/mus_129s1_20220826.png) [129S1](https://www.jax.org/strain/002448)
![c57bl6jx1](/img/b6j_20220826193408.png) [C57BL/6J](https://www.jax.org/strain/000664)
![balbcjx1](/img/BALBcj_20220826194024.png) [BALB/cJ](https://www.jax.org/strain/000651)

### Mouse genetics tools in brain vasculature system field
1. **NG2DsRedBAC**: NG2DsRedBAC transgenic mice is suitable for pericyte and Artery SMC labeling in central nervous system. NG2DsRedBAC transgenic mice express an optimized red fluorescent protein variant (DsRed.T1) under the control of the mouse NG2 (Cspg4) promoter/enhancer and may be useful for fluorescent labeling of NG2 cells (oligodendrocyte progenitor cells) in the central nervous system and NG2-expressing cells in other organs, as well as for isolating NG2 cell populations via FACS. [[JAX Strain #:008241](https://www.jax.org/strain/008241) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=008241&protocolID=18794)] | [DOI Link](https://doi.org/10.1242/dev.004895)]
2. **Tek-Cre**: The Tie2-Cre (Tek-Cre) transgene has the mouse endothelial-specific receptor tyrosine kinase (Tek or Tie2) promoter directing expression of Cre recombinase. These Tek-Cre transgenic mice are a Cre-lox tool useful for deletion of floxed sequences in endothelial cells during embryogenesis and adulthood. **Cre recombinase activity is also detected in microglia in central nervous system.**  [[JAX Strain #:008863](https://www.jax.org/strain/008863) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=008863&protocolID=22392) | [DOI Link](https://doi.org/10.1006/dbio.2000.0106)]
3. **Cdh5-PAC-CreERT2**: [[Genotyping Protocol](https://static1.squarespace.com/static/542b27b6e4b04193388466d4/t/5432b54fe4b0233b902a4d78/1412609359339/Cdh5+%28PAC%29+CreERT2.pdf) | [DOI Link](https://doi.org/10.1038/nature09002)]
4. **Pdgfrb-cre**: Pericyte labeling. [DOI Link](https://doi.org/10.1002/dvg.20769)
5. **Pdgfrb-CreERT2**: Pdgfrb-CreERT2 transgenic mice is suitable for Pericyte labeling, the result of Pdgfrb-CreERT2::Ai14 is very clean and specific. [[JAX Strain #:029684](https://www.jax.org/strain/029684) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=029684&protocolID=20745) | [DOI Link](https://doi.org/10.1038/ki.2015.274)]
6. **BMX-CreERT2**: Artery EC labeling.  [[Genotyping Protocol](https://static1.squarespace.com/static/542b27b6e4b04193388466d4/t/58ee4b7b9f745687d35f380f/1492011904519/Tg+%28BMX-CreERT2%29+Genotyping.pdf)] 
7. **Slco1c1-KI-P2A-iCreERT2**: Brain specific EC labeling(including spinal cord). [Woo-Ping Ge lab, Design by Jun-Liszt Li, [Contact](https://www.cibr.ac.cn/science/team/detail/404?language=en)]

8. 




### Astrocyte labeling
1. **Aldh1l1-EGFP**: In this transgenic strain, Aldh1l1 promoter-active cells in the brain and spinal cord (the majority being astrocytes) express EGFP. Cre-mediated excision of the floxed EGFP/Stop enables DTA-mediated ablation of these cells. [[JAX Strain #:026033](https://www.jax.org/strain/026033) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=026033&protocolID=32517) | [PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERxxhv6tqNJJhC7-Zh2Ed10B7Abd5J_NvmqkXRpNZHZt8A?e=tuywZB) | [PMID: 22745251](https://pubmed.ncbi.nlm.nih.gov/22745251/) | [DOI Link](https://doi.org/10.1126/science.1222381)]
2. **Aldh1l1-CreERT2**: Aldh1l1-Cre/ERT2 BAC transgenic mice have tamoxifen-inducible Cre recombinase expression directed at high levels to the vast majority of astrocytes, with no detectable expression in neurons. These mice allow pan-astrocytic, specific and inducible genetic manipulations in vivo for studying astrocyte biology at different developmental stages in neural circuitry/synapses, behavior, disease and injury/trauma. [[JAX Strain #:029655](https://www.jax.org/strain/029655) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=029684&protocolID=20745)]
3. **mGFAP-Cre**
4. **hGfap-CreERT2**:
    - Park, Yongmin Mason, et al. "Astrocyte specificity and coverage of hGFAP-CreERT2 [Tg (GFAP-Cre/ERT2) 13Kdmc] mouse line in various brain regions." Experimental Neurobiology 27.6 (2018): 508. [[PMID: 30636902](https://pubmed.ncbi.nlm.nih.gov/30636902/) | [DOI Link](https://doi.org/10.5607/en.2018.27.6.508)]

5. **Glast-CreERT2**
6. **Fgfr3-iCreERT2**




### Neuron labeling
1. **Thy1-YFP-M**: These Thy1-GFP-M transgenic mice may be useful in neurobiological studies for fluorescent labeling of neural tissues, especially for mossy fibers in the cerebellum and intense, yet sparse, labeling of a variety of neuronal subsets. [[JAX Strain #:007788](https://www.jax.org/strain/007788) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=007788&protocolID=25728)]
2. **Thy1-YFP-16J**: These thy1-YFP-16 transgenic mice express yellow fluorescent protein at high levels in motor and sensory neurons, as well as in subsets of central neurons. This line provides a strong and specific vital marker for axons, and expression is strong from a mid-gestational stage into adulthood. [[JAX Strain #:003709](https://www.jax.org/strain/003709) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=003709&protocolID=23870)]
2. **Gad1-EGFP**: Also called Gad67-EGFP, Inhibitory GABAnergic neuron labeling. [[JAX Strain #:007677](https://www.jax.org/strain/007677) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=007677&protocolID=22102) | [DOI Link](https://doi.org/10.1523/jneurosci.1851-04.2004)] 
3. **Slc32a1-IRES2-FlpO**: Slc32a1-IRES2-FlpO-D knock-in mice are designed to have optimized FLP recombinase expression directed to VGAT-expressing cells. FlpO expression/activity largely recapitulates the endogenous Slc32a1 gene in the brain - scattered Cre expression is observed throughout many brain regions, including the cortex, hippocampus, striatum and thalamus. These mice may be used to generate conditional mutations for studying gain-or-loss of function and/or fate mapping related to GABAergic neuronal function. [[JAX Strain #:031331](https://www.jax.org/strain/31331) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=031331&protocolID=33044) | [PMID: 30007418](https://pubmed.ncbi.nlm.nih.gov/30007418/) | [DOI Link](https://doi.org/10.1016/j.cell.2018.06.035)]
4. **Vip-Cre**: Vip+ Interneuron labeling.
5. **Sst-Cre**: Sst+ Interneuron labeling.
6. **Som-Cre**: Som+ Interneuron labeling.
7. **PV-Cre**: PV+ Interneuron labeling.
7. **CamKIIa-Cre**: Random insertion, Expression pattern in brain(Throughout the cortex, hippocampus, striatum, and other structures) | [Susumu Tonegawa] | 
8. **Snap25-IRES2-Cre**
9. **Slc17a7-IRES2-Cre**: Slc17a7 (VGlut1), Excitatory neuron labeling [[PMID: 25071457](https://pubmed.ncbi.nlm.nih.gov/25071457/) | [DOI Link](https://doi.org/10.3389/fncir.2014.00076)]
10. **Drd1-Cre**: These mice express Cre recombinase under the control of the mouse Drd1a (dopamine receptor D1A) promoter. DRD1A is s a G-protein coupled receptor involved in the regulation of neuronal growth and development. The D1 subtype is the most abundant dopamine receptor in the central nervous system. Homozygotes are viable and fertile. [[MMRRC Strain #037156-JAX](https://www.jax.org/strain/024860) | [DOI Link](https://doi.org/10.1523/jneurosci.3795-06.2006)]
11. **Drd2-Cre**


### Microglia labeling
1. **CX3CR1-EGFP**: [[JAX Strain #:005582](https://www.jax.org/strain/005582) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=005582&protocolID=27927)]
2. **CX3CR1-iCreERT2**:
3. **Tmem119-P2A-iCreERT2**:  [[JAX Strain #:031820](https://www.jax.org/strain/031820) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=031820&protocolID=34019) | [DOI Link](https://doi.org/10.1523/eneuro.0448-18.2019)]


### Oligodendrocytes
1. **Oligo2-Cre**
2. **Plp1-CreERT2**: These transgenic mice have a tamoxifen inducible Cre-mediated recombination system driven by the mouse Plp1, proteolipid protein (myelin) 1 promoter. When crossed with a strain containing a loxP site flanked sequence of interest, the offspring are useful for generating tamoxifen-induced, Cre-mediated targeted deletions. Tamoxifen administration allows for ablation of predetermined genes in oligodendrocytes and Schwann cells. [[JAX Strain #:005975](https://www.jax.org/strain/005975) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=005975&protocolID=43081) | [DOI Link](https://doi.org/10.1002/gene.10154)]

### Precusor cells



### Reporter Lines
1. **Ai14(Rosa26-CAG-LSL-tdTomato-WPRE-pA)**
2. **Ai47(Rosa26-CAG-LSL-emerald-TagGFP-hrGFP-WPRE-pA)**: The Rosa-CAG targeting vector was designed with (from 5' to 3') a CMV-IE enhancer/chicken beta-actin/rabbit beta-globin hybrid promoter (CAG), an FRT site, a loxP-flanked STOP cassette (with stop codons in all 3 reading frames and a triple polyA signal), triple GFP reporter genes (emerald, TagGFP (Aequorea macrodactyla GFP), and hrGFP (Humanized Renilla reniformis GFP)), a woodchuck hepatitis virus post-transcriptional regulatory element (WPRE; to enhance the mRNA transcript stability), a polyA signal, and an attB/attP-flanked PGK-FRT-Neo-polyA cassette. This entire construct was inserted between exons 1 and 2 of the Gt(ROSA)26Sor locus. [[MGI:5750793](http://www.informatics.jax.org/allele/MGI:5750793) | [PMID: 30007418](https://pubmed.ncbi.nlm.nih.gov/30007418/) | [DOI Link](https://doi.org/10.1016/j.cell.2018.06.035)]
3. **Ai140(TIT2L-EGFP-ICL-tTA2)**: Ai140 (also called Ai140(TIT2L-EGFP-ICL-tTA2)) is a Cre-dependent, Tet-controllable fluorescent reporter line, created by targeted insertion at the Igs7 locus (TIGRE). Exposure to Cre recombinase removes both STOP cassettes - resulting in tTA2 expression and robust EGFP fluorescence. EGFP expression may be expected to be diminished/eliminated by doxycycline. [[JAX Strain #:034100](https://www.jax.org/strain/034100) | [Genotyping Protocol](https://www.jax.org/Protocol?stockNumber=034100&protocolID=38356) | [PMID: 30007418](https://pubmed.ncbi.nlm.nih.gov/30007418/) | [DOI Link](https://doi.org/10.1016/j.cell.2018.06.035)]
4. 
5.




### Transgenic line for Calcium imaging
1. Thy1-GCaMP6f
2. Thy1-JRGECO1a
3. Ai96(Rosa26-CAG-LSL-GCaMP6s)




### Transgenic line for genome editing
1. Rosa26-CAG-LSL-SpCas9-P2A-EGFP
2. Rosa26-CAG-SpCas9-P2A-EGFP





### Ref. public websites
1. [The Jackson Laborary](https://www.jax.org/): The laboratory is also the world's source for more than 8,000 strains of genetically defined mice, and serves as a worldwide center for scientific courses, conferences, training, and education.
2. [Mouse Genome Informatics](http://www.informatics.jax.org/): MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.
3. [Allen Institute for Brain Research](http://connectivity.brain-map.org/transgenic/)


### Ref. Labs
1. [Susumu Tonegawa Lab](https://tonegawalab.mit.edu/susumu-tonegawa/), MIT
2. [Joe Z. Tsien Lab](https://loop.frontiersin.org/people/280979/overview), Princeton University
- **Subregion- and Cell Type–Restricted Gene Knockout in Mouse Brain with phage P1–derived Cre/loxP recombination system**
    - Tsien, Joe Z., et al. "Subregion-and cell type–restricted gene knockout in mouse brain." Cell 87.7 (1996): 1317-1326. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ee8o1FlF7rlLtenwMNQHY2MBe1rvvTrgiYrFzTkghoxrWA?e=0WhPdu) | [PMID: 8980237](https://pubmed.ncbi.nlm.nih.gov/8980237/) | [DOI Link](https://doi.org/10.1016/S0092-8674(00)81826-7)]
3. [Masashi Yanagisawa Lab](https://sleepymouse.jp/), The International Institute for Integrative Sleep(IIIS)
4. [Z. Josh Huang Lab](https://www.neuro.duke.edu/research/faculty-labs/huang-lab), Duke University


### Ref. Protocols
1. 



---
# Rat






---
# Drosophila melanogaster
![seymour_benzer1](/img/Seymour_and_thefly.png)
Seymour Benzer with mega-Drosophila, 1974.
- **Seymour Benzer(1921-2007)**, California Institute of Technology [[Public Interview](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/ESTLH4bQeIVEobzIprKewX0BtTvgmcitTEOq1JAt_DVJBw?e=peh8MA) |  [Read the Interview](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EZVmcpluPh9Pkayf_EYVUugB8Q9QW5Y5qCPsVW00Of3z7w?e=faAcdk)] | [Pioneer works: [phage genome structure mapping, 1959, PNAS](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EanvBkFSnedAtSTtSqkzp08B12weMOgnka7hOQbo4FJFyg?e=Xa00vZ) | [Clock Mutants of Drosophila melanogaster, 1971, PNAS](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ebu6ooZTaflCk6zzq4LdNLkB51wPJwNB8mGrdCY4ACm47g?e=2NXNPL)]



### Ref. Websites
1. [FlyBase](https://flybase.org/): A Database of Drosophila Genes & Genomes

---
# Zebrafish



### Ref. Websites
1. [ZFIN](https://zfin.org/): The Zebrafish Information Network (ZFIN) is the database of genetic and genomic data for the zebrafish (Danio rerio) as a model organism. ZFIN provides a wide array of expertly curated, organized and cross-referenced zebrafish research data.



---
## Nobel laureates
- **Michael W. Young**, Rockefeller University, New York, NY, USA\
The Nobel Prize in Physiology or Medicine 2017\
**Time Travels: A 40 Year Journey from Drosophila’s Clock Mutants to Human Circadian Disorders** [[Nobel Lecture video](https://www.youtube.com/watch?v=Blm3QwT-rRk&ab_channel=NobelPrize) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EQ3Lh8WjagRFoiNVbUsIbk0BbkzwPzSe2gswYWApBQQSoA?e=70flib) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERsnFw1YSGpKmK-x6I-OcY8B_SnYvmbc9TuU_qTaM_bA4Q?e=GYkHk2) | [Source](https://www.nobelprize.org/prizes/medicine/2017/young/facts/)]

- **Sydney Brenner(1927-2019)**, The Molecular Sciences Institute, Berkeley, CA\
The Nobel Prize in Physiology or Medicine 2002\
**Nature's Gift to Science** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/Ec3SUiYnkIZAnC5VfbVEuQcBCIB9298kRxmK85J5uhUt0Q?e=Lh7u48) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERaf-DRSrGVOkrs3delRx44B9TsHkEVqe2q1iZhgTmUdgg?e=8xBNmp) | [Source](https://www.nobelprize.org/prizes/medicine/2002/brenner/biographical/)]

- **Barbara McClintock(1902-1992)**, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA\
The Nobel Prize in Physiology or Medicine 1983\
**The Significance of Responses of the Genome to Challenge** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/Ec3SUiYnkIZAnC5VfbVEuQcBCIB9298kRxmK85J5uhUt0Q?e=Lh7u48) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERsnFw1YSGpKmK-x6I-OcY8B_SnYvmbc9TuU_qTaM_bA4Q?e=GYkHk2) | [Source](https://www.nobelprize.org/prizes/medicine/1983/mcclintock/facts/)]

- **Max Delbrück(1906-1981)**, California Institute of Technology (Caltech), Pasadena, CA, USA\
The Nobel Prize in Physiology or Medicine 1969\
**A Physicist’s Renewed Look at Biology–Twenty Years Later** [[Read the Lecture](https://www.nobelprize.org/prizes/medicine/1969/delbruck/lecture/) | [Source](https://www.nobelprize.org/prizes/medicine/1969/delbruck/facts/)]

- **Alfred D. Hershey(1908-1997)**, Carnegie Institution of Washington, Long Island, New York, NY, USA\
The Nobel Prize in Physiology or Medicine 1969\
**Idiosyncrasies of DNA Structure** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EUf33ViYqKJGu4HB54BrW_0BN0V8yZyfuIVh0TPC6dN27g?e=UFffO9) | [Source](https://www.nobelprize.org/prizes/medicine/1969/hershey/facts/)]

- **Salvador E. Luria(1912-1991)**, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA\
The Nobel Prize in Physiology or Medicine 1969\
**Phage, colicins and macroregulatory phenomena** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EeAqxSmGYFtKmsp1Wq-WrQMBjwfHLohdJ2-wTmZxbWAzrg?e=nDVf7X) | [Source](https://www.nobelprize.org/prizes/medicine/1969/luria/facts/)]

- **Thomas Hunt Morgan(1866-1945)**, California Institute of Technology (Caltech), Pasadena, CA, USA\
The Nobel Prize in Physiology or Medicine 1933\
**The Relation of Genetics to Physiology and Medicine** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EZA1WEVd_41Psh2ixX6hStgBn8ZS5mXL0nDCH9nD4ybNdg?e=P3yMPE) | [Source](https://www.nobelprize.org/prizes/medicine/1933/morgan/facts/)]







