---
title: "Genomics tools"
date: 2022-08-21T10:43:59+08:00
categories:
- category
- subcategory
tags:
- RNA-seq
- scRNAseq
- NGS
keywords:
- tech
comments:       false
showMeta:       false
showActions:    false
thumbnailImage: /img/genomics_NGSx1.png
thumbnailImagePosition: left
#thumbnailImage: //example.com/image.jpg
---

# RNA-Seq

> 

#### KITS FOR RNA EXTRACTION
Takara: Smart-Seq


#### SEQUENCING PLATFORM
[illumina](https://www.illumina.com/);
[Huada Gene](https://www.genomics.cn/sequecplatform.html)



#### ANALYSIS SOFTWARES

**QC**\
[fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/);
[multiqc](https://multiqc.info/);
[fastp](https://github.com/OpenGene/fastp)



**Reads Mapping**\
[HISAT2](http://daehwankimlab.github.io/hisat2/): graph-based alignment of next generation sequencing reads to a population of genomes. ([Manual](http://daehwankimlab.github.io/hisat2/manual/))\
[STAR](https://github.com/alexdobin/STAR): The STAR software package maps large sets of high-throughput sequencing reads to a reference genome with high levels of accuracy and speed. ([Manual](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EWzT0NuF1ANJsMKXYwCwuNYBIRXeIC-AcJ-hx6XSar7Ebw?e=j0P6Rs))\
[TopHat](http://ccb.jhu.edu/software/tophat/index.shtml): TopHat is a fast splice junction mapper for RNA-Seq reads. It aligns RNA-Seq reads to mammalian-sized genomes using the ultra high-throughput short read aligner Bowtie, and then analyzes the mapping results to identify splice junctions between exons. ([Manual](http://ccb.jhu.edu/software/tophat/manual.shtml))



Samtools:


**Gene Counting**\
FeatureCounts;
HTseq-Count:


**Difference Analysis**\
R packages: DESeq2; edgeR;


**Single-Cell RNAseq**\
R packages: [Seurat](https://satijalab.org/seurat/)



**Gene Symbol Converter**:\
https://david.ncifcrf.gov/content.jsp?file=conversion.html
https://www.syngoportal.org/convert.html





**Enrichment analysis tools**\
http://www.pantherdb.org/


**Genome resource**\
[UCSC Genome Browser](http://genome.ucsc.edu/):
[NCBI Mouse Resources](https://www.ncbi.nlm.nih.gov/genome?term=mus%20musculus):


**Gene name search**\
[MGI](http://www.informatics.jax.org/marker/MGI:2685385):
[Uniprot](https://www.uniprot.org/):


**Brain Atlas from Allen Institute for Brain Science(AIBS)**\
http://celltypes.brain-map.org/rnaseq/mouse/cortex-and-hippocampus


**Single Cell Sequencing Platforms**
10X genomics;


**Transcriptome database of different Brain cell types**
1. https://www.brainrnaseq.org/, Started at [Ben A. Barre lab](), maintained by [Ye Zhang lab](http://www.yezhanglab.org/yezhang).


**Brain single-cell genomics Browsers**

1. Dropviz: http://dropviz.org/

2. Mousebrain (Both adolescent and development): http://mousebrain.org/

3. VascularSingleCell: http://betsholtzlab.org/VascularSingleCells/database.html

4. Bulk RNAseq of different Brain cell types: https://www.brainrnaseq.org/





**Ref papers**:

1. Sahraeian, Sayed Mohammad Ebrahim, et al. "Gaining comprehensive biological insight into the transcriptome by performing a broad-spectrum RNA-seq analysis." Nature communications 8.1 (2017): [Link](https://www.nature.com/articles/s41467-017-00050-4)
2. 


#### Ref labs
1. **Single cell genomics**: [Satija lab, NYU](https://satijalab.org/)\
2. **Spatial Transcriptomics**: [Rongfan lab, Yale](https://www.eng.yale.edu/fanlab/Responsive_Fan_lab/index.html)
3. **Transcriptomics of brain cell types**: 
- **Ben A. Barre(1954-2017)**, Stanford University School of Medicine\
    Pioneer of glia biology
    [A Tribute to Ben Barres]()
    - Cahoy, J. D. et al. A Transcriptome Database for Astrocytes, Neurons, and Oligodendrocytes: A New Resource for Understanding Brain Development and Function. Journal of Neuroscience 28, 264â€“278 (2008). [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EdtWpOFux75DtCYSoiM3UuUBr-4OU0A1XKl1UBCtVFcWKA?e=o9ZFRu) | [PMID: 18171944](https://pubmed.ncbi.nlm.nih.gov/18171944/) | [DOI Link](https://doi.org/10.1523/jneurosci.4178-07.2008)]


---
## Nobel laureates
1. **Frederick Sanger(1918-2013)**, Cambridge University, Great Britain\
The Nobel Prize in Chemistry 1980\
**Determination of Nucleotide Sequences in DNA**
[[Public talk](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/ESLmFc2bPj1NiQLr8MynjR8ByJViMi7kZA4tovrSNIpedg?e=4Bclbz) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EZMCejlGVNNGkWrDQA9Ex4sBUyUIoGYRkafoFt2BA7T3SQ?e=Js9jO3) |  [Source](https://www.nobelprize.org/prizes/chemistry/1980/sanger/facts/)]

2. **Walter Gilbert(1932-)**, Harvard University, Biological Laboratories, Cambridge, MA, USA\
The Nobel Prize in Chemistry 1980\
**DNA Sequencing and Gene Structure**
[[iBiology interview]() | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ESLasGpogvVMvh4lDoG907IB81_uZWP46kxzSYNsvLox8A?e=pWUlZs) | [Source](https://www.nobelprize.org/prizes/chemistry/1980/gilbert/biographical/)]

