---
title: "Molecular Biology"
date: 2022-08-21T20:27:51+08:00
categories:
- category
- subcategory
tags:
- CRISPR
- FP
- Optogenetics
keywords:
- tech
comments:       false
showMeta:       false
showActions:    false
thumbnailImage: /img/pmx090121colgenetics-001-1630015763.jpg
thumbnailImagePosition: left
#thumbnailImage: //example.com/image.jpg
---
> What I cannot create, I do not understand.\
--- Richard P. Feynman

<!--more-->

![quote_feynman](/img/quote_feynman.jpg)



---
## Tissue/Cell type specific AAVs


- **Liver targeting**:


- **Brain endothelial cell targeting**: 
    - AAV-BR1: Neurons(Cortex, Striatum, Cerebellum); Brain endothelial cells(Whole brain and spinal cord); has central nervous system specificity. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERka7Rr3iDhLhVZTKh8dj7ABHgOXCwzpN2avX_XUSP8jvw?e=0AOQ70) | [PMID: 27137490](https://pubmed.ncbi.nlm.nih.gov/27137490/) | [DOI Link](https://doi.org/10.15252/emmm.201506078)]
    - AAV-BI30: AAV9 variants, with liver leakage. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EUxMCsFh91tDl49LHoAorpkBT1REK6IuHFHofD7ze-dcFA?e=Z0s0iN) | [PMID: 35571675](https://pubmed.ncbi.nlm.nih.gov/35571675/) | [DOI Link](https://doi.org/10.1038/s44161-022-00046-4)]

- **Microglia targeting**: 
    - AAV2/6(Y731F/Y705F/T492V) [[PMID: 27308302](https://pubmed.ncbi.nlm.nih.gov/27308302/) | [DOI Link](https://doi.org/10.1038/mtm.2016.26)]
    - AAV-MG[AAV-cMG] [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ee-bqHwpqqJIuiYwXlCAJSMBPOUjYvUN59IiY6MLf8tVtg?e=iUmxnI) | [PMID: 35879607](https://pubmed.ncbi.nlm.nih.gov/35879607/) | [DOI Link](https://doi.org/10.1038/s41592-022-01547-7)]

- **Astrocyte targeting**:
    - AAV-PhP.B5
    - AAV-PhP.AX [[]]

- **Brain organ specific**: 
    - AAV-BR1: Based on AAV2 wt cap, screen and recovered by brain homing assay. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERka7Rr3iDhLhVZTKh8dj7ABHgOXCwzpN2avX_XUSP8jvw?e=0AOQ70) | [PMID: 27137490](https://pubmed.ncbi.nlm.nih.gov/27137490/) | [DOI Link](https://doi.org/10.15252/emmm.201506078)]



- **Global/non-specific**: 
    - AAV-PhP.B: AAV-PHP.B transfers genes throughout the CNS with an efficiency that is at least 40-fold greater than that of the current standard, AAV9. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EWltFqorabFEuOGjnhUXIs4BJUwlIvtdn0gvR-D5SiYljw?e=7fGVFB) | [PMID: 26829320](https://pubmed.ncbi.nlm.nih.gov/26829320/) | [DOI Link](https://doi.org/10.1038/nbt.3440)]

    - AAV-PhP.eB: [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ee3jKWcTBfNEmGVr907iDgUBvkTOtwgl7KyYe5ic2b56bg?e=dkOFlB) | [PMID: 35879607](https://pubmed.ncbi.nlm.nih.gov/28671695/) | [DOI Link](https://doi.org/10.1038/nn.4593)]


- **Peripheral nervous system**:
    - AAV-PHP.S: 1 × 10^12 vg of AAV-PHP.S transduced 82% of dorsal root ganglion neurons, as well as cardiac and enteric neurons. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ee3jKWcTBfNEmGVr907iDgUBvkTOtwgl7KyYe5ic2b56bg?e=dkOFlB) | [PMID: 35879607](https://pubmed.ncbi.nlm.nih.gov/28671695/) | [DOI Link](https://doi.org/10.1038/nn.4593)]


## Tissue/Cell type specific promoters


- **Pan-neuron**: 
    - hSyn: Kügler, S., E. Kilic, and M. Bähr. "Human synapsin 1 gene promoter confers highly neuron-specific long-term transgene expression from an adenoviral vector in the adult rat brain depending on the transduced area." Gene therapy 10.4 (2003): 337-347. [[PDF]() | [DOI Link](https://doi.org/10.1038/sj.gt.3301905) ]

- **Excitatory neurons**: 
    - CaMKIIa: Sunyer, T. & Sahyoun, N. Sequence analysis and DNA-protein interactions within the 5’ flanking region of the Ca2+/calmodulin-dependent protein kinase II alpha-subunit gene. Proc Natl Acad Sci U S A 87, 278–82 (1990). [[PDF]() | | [PMID: 2153289](https://pubmed.ncbi.nlm.nih.gov/2153289/) | [DOI Link](https://doi.org/10.1073/pnas.87.1.278)]
    

- **Inhibitory neurons**: 
    - pGAD1 
    - mDlx

- **Interneurons**
    - **SST neurons**: pSST [[PMID: 18391943](https://pubmed.ncbi.nlm.nih.gov/18391943/)]
    - **Cholecystokinin(CCK) neurons**: pCCK [[PMID: 19164102](https://pubmed.ncbi.nlm.nih.gov/19164102/)]

- **Purkinje Cell-specific**: PCP2/L7

- **TH+ Neurons**: pTH(Rat), pTH(Mus)


- **Hypocretin neurons**: Hypocretin(Hcrt, also known as orexin), mHcrt Pro[[PMID: 17082459](https://pubmed.ncbi.nlm.nih.gov/17082459/) | [PMID: 10364220](https://pubmed.ncbi.nlm.nih.gov/10364220/)]

- **Melanin concentrating hormone neurons**: ppMCH [[PMID: 15157424](https://pubmed.ncbi.nlm.nih.gov/15157424/)]

- **Astrocyte**: GfABC1D: short GFAP promoter[]

- **Microglia**: Cd68[]

- **Endothelial cells**: 

- **Liver hepatocyte**: P3[]

- **Constitutive expression**: EF1a, CAG


## Activity response promoters:

- **Drug induced**: Tet-on system

- **Activity dependent**: c-Fos

- **ERK pathway activation**: 
    - pKrox24 [[PMID: 28199182](https://pubmed.ncbi.nlm.nih.gov/28199182/) | [DOI Link](https://doi.org/10.7554/elife.21536)]
    - ERK-KTR: Kinase translocation reporters(KTRs) enable multiplexed measurements of the dynamics of kinase activity at a single-cell level. [[PMID: 29266096](https://pubmed.ncbi.nlm.nih.gov/29266096/) | [DOI Link](https://doi.org/10.1038/nprot.2017.128)]




---
## Fluorescence Proteins

- NIR FPs: 
  - **miRFP670nano**: Unlike BphP-derived NIR FPs, miRFP670nano is highly stable to denaturation and degradation and can be used as an internal protein tag. miRFP670nano is an effective FRET donor for red-shifted NIR FPs, enabling engineering NIR FRET biosensors spectrally compatible with GFP-like FPs and blue-green optogenetic tools. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EUyyrwoLIGhLmPy3YtH0ExEBal0dkC8RTA-0-Syq7q7-2A?e=8F9zoB) | [PMID: 30655515](https://pubmed.ncbi.nlm.nih.gov/30655515/) | [DOI Link](https://doi.org/10.1038/s41467-018-08050-8)]
  - **miRFP670nano3**: The enhanced miRFP670nano3 protein fluoresces at excitation and emission wavelengths of 645 nm and 670 nm, respectively, and its spectral properties are similar to those of the parental miRFP670nano. miRFP670nano3 has a molecular brightness (product of molar extinction coefficient and quantum yield) **twice as high as** that of miRFP670nano and **1.5 times higher** than that of the widely used red FPs mCherry or mCardinal. The extinction coefficient (129,000 M-1 cm-1) and fluorescence quantum yield (18.5%) of miRFP670nano3 are **the highest among all monomeric NIR FPs** with BV chromophores. [[Addgene Plasmid: #184668](https://www.addgene.org/184668/) |  [PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EUyyrwoLIGhLmPy3YtH0ExEBal0dkC8RTA-0-Syq7q7-2A?e=sWUdzi) | [PMID: 35606446](https://pubmed.ncbi.nlm.nih.gov/35606446/) | [DOI Link](https://doi.org/10.1038/s41592-022-01467-6)]
  - **miRFP718nano**: miRFP718nano efciently binds endogenous biliverdin chromophore and brightly
fuoresces in mammalian cells and tissues. miRFP718nano has maximal emission at 718 nm and an emission tail in the short-wave infrared (SWIR) region, allowing deep-penetrating of-peak fuorescence imaging in vivo. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Ef476-SLNONImc8HX00Ii7kBftzF3asjtA6P1n4mQzU0UA?e=wh8VH4) | [PMID: 36456785](https://pubmed.ncbi.nlm.nih.gov/36456785/) | [DOI Link](https://doi.org/10.1038/s41592-022-01683-0)]


- RFPs:  
  - **mCherry2**: mCherry2 is a basic (constitutively fluorescent) red fluorescent protein published in 2017[Shen, Yi, et al. PLoS One], derived from Discosoma sp.. It has very low acid sensitivity. [[Addgene Plasmid: #54800](https://www.addgene.org/54800/) | [PMID: 28241009](https://pubmed.ncbi.nlm.nih.gov/28241009/) | [DOI Link](https://doi.org/10.1371/journal.pone.0171257)]
  - **mCardinal**: mCardinal is a basic (constitutively fluorescent) far red fluorescent protein published in 2014[Chu, Jun, et al Nature methods 2014], derived from Entacmaea quadricolor. It is reported to be a rapidly-maturing monomer with moderate acid sensitivity. [[Addgene Plasmid: #54800](https://www.addgene.org/54800/) | [PMID: 24633408](https://pubmed.ncbi.nlm.nih.gov/24633408/) | [DOI Link](https://doi.org/10.1038/nmeth.2888)]
  - **mRuby3**：mRuby3 is a basic (constitutively fluorescent) red fluorescent protein published in 2016, derived from Entacmaea quadricolor. It has low acid sensitivity.
[[Addgene Plasmid: #74234](https://www.addgene.org/74234/) | [PMID: 26879144](https://pubmed.ncbi.nlm.nih.gov/26879144/) | [DOI Link](https://doi.org/10.1038/srep20889)]
  - **stagRFP**: A new single-residue mutant, super-TagRFP (stagRFP) has nearly twice the molecular brightness of TagRFP-T and negligible photoactivation. [[PMID: 32296061](https://pubmed.ncbi.nlm.nih.gov/32296061/) | [DOI Link](https://doi.org/10.1038/s41467-020-15687-x)]
  - **mScarlet**:
  - **mScarlet3**:
  


- GFPs: 
  - **EGFP**: EGFP is a basic (constitutively fluorescent) green fluorescent protein published in 1996, derived from Aequorea victoria. It is reported to be a rapidly-maturing weak dimer with moderate acid sensitivity. [[FPbase](https://www.fpbase.org/protein/egfp/) | 
[PMID: 8707053](https://pubmed.ncbi.nlm.nih.gov/8707053/) | [DOI Link](https://doi.org/10.1016/0378-1119(95)00685-0)]
  - **mGreenLantern**: mGreenLantern is a basic (constitutively fluorescent) green fluorescent protein published in 2020[Campbell, Benjamin C., et al. PNAS], derived from Aequorea victoria. It is reported to be a very rapidly-maturing monomer with moderate acid sensitivity. **mGreenLantern was 190% brighter than mNeonGreen, 260% brighter than Clover, and 620% brighter than EGFP when expressed in HeLa cells and showed faster and complete maturation.**
[[Addgene Plasmid: #164469](https://www.addgene.org/164469/) | [PMID: 33208539](https://pubmed.ncbi.nlm.nih.gov/33208539/) | [DOI Link](https://doi.org/10.1073/pnas.2000942117)]
  - **StayGold**: StayGold, a green fluorescent protein (GFP) derived from the jellyfish Cytaeis uchidae. StayGold is over one order of magnitude more photostable than any currently available fluorescent protein and has a cellular brightness similar to mNeonGreen. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EWDddVtu-BxMnGXCVXGWtGEBbvSlIS9zU6n8lONjE4uKYQ?e=HOQnUc) | [PMID: 35468954](https://pubmed.ncbi.nlm.nih.gov/35468954/) | [DOI Link](https://doi.org/10.1038/s41587-022-01278-2)]


- YFPs: 
  - **dLanYFP**: dLanYFP is a basic (constitutively fluorescent) green/yellow fluorescent protein published in 2013, derived from Branchiostoma lanceolatum. [[Addgene Plasmid: #164469]() | [PMID: 23524392](https://pubmed.ncbi.nlm.nih.gov/23524392/) | [DOI Link](https://doi.org/10.1038/nmeth.2413)]
  - **tdLanYFP**: tdLanYFP, derived from the tetrameric protein from the cephalochordate B. lanceolatum, LanYFP. Contrasting with EYFP and its derivatives, tdLanYFP has a very high photostability in vitro and in live cells. As a consequence, tdLanYFP allows imaging of cellular structures with sub-diffraction resolution using STED nanoscopy and is compatible with the use of spectro-microscopies in single molecule regimes. [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EfWLZ4vhkepOrKXwwTKIFpsBI_15ugNDSw613yqidDq4aA?e=ofNQ7S) | 
[DOI Link](http://dx.doi.org/10.1021/acssensors.1c00874)]
  - **hfYFP**: ‘hyperfolder YFP’ (hfYFP), hfYFP contains no cysteines, is chloride insensitive and tolerates aldehyde and osmium tetroxide fixation better than common fluorescent proteins, enabling its use in expansion and electron microscopies. [[PMID: 36344833](https://pubmed.ncbi.nlm.nih.gov/36344833/) | [DOI Link](https://doi.org/10.1038/s41592-022-01660-7)]



- BFPs: 
  - **TagBFP**: TagBFP is a basic (constitutively fluorescent) blue fluorescent protein published in 2008, derived from Entacmaea quadricolor. It is reported to be a very rapidly-maturing monomer with very low acid sensitivity. [[FPbase](https://www.fpbase.org/protein/tagbfp/) | [PMID: 18940671](https://pubmed.ncbi.nlm.nih.gov/18940671/) | [DOI Link](https://doi.org/10.1016/j.chembiol.2008.08.006)]
  - **mTagBFP2**: mTagBFP2 is a basic (constitutively fluorescent) blue fluorescent protein published in 2011, derived from Entacmaea quadricolor. It is reported to be a very rapidly-maturing monomer with very low acid sensitivity. [[FPbase](https://www.fpbase.org/protein/mtagbfp2/) | [PMID: 22174863](https://pubmed.ncbi.nlm.nih.gov/22174863/) | [DOI Link](https://doi.org/10.1371/journal.pone.0028674)]




## Genetically encoded indicators
**| Pioneer work**: 
    - Engineer of cpEGFP and GCaMP1 [[PMID: 11248055](https://pubmed.ncbi.nlm.nih.gov/11248055/)]
- **Calcium**: GCaMP series [[GCaMP8s/m/f](https://www.janelia.org/jgcamp8-calcium-indicators) | [PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EcWtwJb1nk5Hq3K9u7TelA0Bg6HOZ4OB5_D_0fG75ucM2w?e=tYNu8A) | [DOI Link](https://doi.org/10.1101/2021.11.08.467793)]; [[GCaMP7s/f/b/c](https://doi.org/10.1038/s41592-019-0435-6)];  [GCaMP6s/m/f](https://doi.org/10.1038/nature12354)]
- **Potassium**: GINKO1: dicator of K+ for optical imaging [[PMID: 30652129](https://pubmed.ncbi.nlm.nih.gov/30652129/) | [DOI Link](https://doi.org/10.1038/s42003-018-0269-2)]
- **Glutamate**: iGluSnFR[[PMID: 30377363](https://pubmed.ncbi.nlm.nih.gov/30377363/) | [DOI Link](https://doi.org/10.1038/s41592-018-0171-3)] ;  iGluSnFR3[[DOI Link](https://doi.org/10.1101/2022.02.13.480251)]

**| Engineered neurotransmitter sensors**
- **Gamma-aminobutyric acid(GABA)**: iGABASnFR [[PMID: 31308547](https://pubmed.ncbi.nlm.nih.gov/31308547/) | [DOI Link](https://doi.org/10.1038/s41592-019-0471-2)]
- **Dopamine**: 
  - GRAB-DA[[GRAB-DA1.0](https://doi.org/10.1016/j.cell.2018.06.042) | [GRAB-DA2.0](https://doi.org/10.1038/s41592-020-00981-9)] 
  - dLight[[dLight1.2](https://doi.org/10.1126/science.aat4422), [addgene](https://www.addgene.org/111068/)]
- **Histamine**:
- **eCB**:

**| Engineered neuropeptide sensors**
- **Oxytocin (OT)**:
  - GRAB-OT[[GRAB-OT1.0](https://doi.org/10.1038/s41587-022-01561-2)]


- **Glucose**:
    - iGlucoSnFR: A genetically encoded single-wavelength sensor for imaging cytosolic glucose [[PMID: 34161775](https://pubmed.ncbi.nlm.nih.gov/34161775/) | [DOI Link](https://doi.org/10.1016/j.celrep.2021.109284)]
- **ATP**: 
    - iATPSnFR1.0: A genetically encoded single-wavelength sensor for imaging cytosolic and cell surface ATP [[PMID: 30755613](https://pubmed.ncbi.nlm.nih.gov/30755613/) | [DOI Link](https://doi.org/10.1038/s41467-019-08441-5)]
    - GRABATP1.0: A sensitive GRAB sensor for detecting extracellular ATP in vitro and in vivo [[PMID: 34942116](https://pubmed.ncbi.nlm.nih.gov/34942116/) | [DOI Link](https://doi.org/10.1016/j.neuron.2021.11.027)]
- **NO**:
    - geNOps: novel multicoloured fluorescent quenching-based  probes by fusing a bacteria-derived -binding domain close to distinct fluorescent protein variants. [[PMID: 26842907](https://pubmed.ncbi.nlm.nih.gov/26842907/) | [DOI Link](https://doi.org/10.1038/ncomms10623)]
- **ONOO-**:
    - pnGFP-Ultra: pnGFP-Ultra exhibits high selectivity and sensitivity toward peroxynitrite with an ~110-fold turn-on response. [[PMID: 33581056](https://pubmed.ncbi.nlm.nih.gov/33581056/) | [DOI Link](https://doi.org/10.1016/j.chembiol.2021.01.013)]
- **Formaldehyde (FA)**: 
    - FAsors: HxlR-based FA sensors, FA-responsive transcription factor HxlR from Bacillus subtilis, which shows that HxlR recognizes FA through an intra-helical cysteine-lysine crosslinking reaction at its N-terminal helix α1, leading to conformational change and transcriptional activation. [[PMID: 33495458](https://pubmed.ncbi.nlm.nih.gov/33495458/) | [DOI Link](https://doi.org/10.1038/s41467-020-20754-4)]
    
- **Voltage indicator**: 
  - VSD-based[[ASAP1](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EaqAq7jvyYhMtPz_Iut6PSEBXapEYn5V30MoeToLPwHOuQ?e=hfyjvW)]
  - Rhodopsin-based[[QuasAr1/2/3/6]() | [Archon1/2] ]

- **Kinase activity**: 
  - PKA[]
  - ERK[]


## Protein Protein interaction Probe
- **BiFC**: LC151 + KN151




#### Ref. Labs
1. [Robert E. Campbell Lab](https://www.chem.s.u-tokyo.ac.jp/campbell/?doing_wp_cron=1670243926.4282240867614746093750),  The University of Tokyo(2022-), University of Alberta, Canada(Previous)
2. [Loren Looger Lab](), Janelia farm/UCSD
3. [Michael Z Lin Lab](https://web.stanford.edu/~mzlin/), Stanford
4. [Mark J Schnitzer Lab](https://pyramidal.stanford.edu/), Stanford
5. [Adam E. Cohen Lab](http://cohenweb.rc.fas.harvard.edu/), harvard






#### Ref. websites

1. [FPbase](https://www.fpbase.org/): FPbase is a moderated, user-editable fluorescent protein database designed by microscopists.



## Pioneers
1. **Stephen W. Kuffler(1913-1980)**: Neurophysiologist and founding father of modern neurobiology. Born in Táp, Hungary, on August 24, 1913, he died on October 11, 1980, in Woods Hole, MA, USA, aged 67 years. [**Biography** [PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ESi3sZ3FB_NIr4HXb-VbuoMBtbPska8tvtRzns7Of9twdg?e=f2Vln4)]
    - **Synaptic transmission**
    - **GABA as an inhibitory neurotransmitter**

2. **Solomon H. Snyder**: Snyder's finding of receptors for neurotransmitters and medicines, as well as the characterization of psychotropic agent activities, has resulted in many breakthroughs in molecular neuroscience. [[Talk video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EYM-PFMexBRGpMbUa5lOfSoBJAksIGtWJD3GH_X1otr_GA?e=rjpFaH)] [**Biography: A Life of Neurotransmitters** [PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERD2vU-Uo7VLuxI0ldTjXK8BChBVhpds_TwEAjjV4bbZmg?e=2BeB1c)]
    - **Co-discovered the opioid receptor** and later identified the existence of normally occurring opiate-like peptides in the brain.
    - **Characterization of novel neurotransmitters**, like the gases NO and CO and the D-isomers of amino acids, including D-serine.
    - **Co-founder of Nova Pharmaceuticals and Guilford Pharmaceuticals**



## CRISPR systems
> The CRISPR-Cas system is a natural immune system for prokaryotes. Some bacteria, after being invaded by a virus, can store a small segment of the virus gene in a storage space called CRISPR in their own genome. When a virus invades again, the bacteria can recognize the virus based on the stored fragment and disable it by cutting off the virus' DNA.
---

### DNA target Cas9 based
> Because of the simplicity with which gRNAs may be designed and the capacity to change practically any genomic locus, CRISPR is a powerful tool for genetic screening investigations.
---

- **Knock out**: SpCas9, SaCas9
- **DNA base editor**: dCas9-xxx
- **Transcriptional activator**: dCas9-Suntag system

- **Cas9 based whole-genome scale pool screen**
    - **CRISPR Pooled Libraries**: Pooled CRISPR libraries are made up of hundreds of plasmids, each holding several gRNAs for each target gene. Target cells are treated with the pooled library in a CRISPR screening experiment to generate a population of mutant cells, which are then screened for a phenotypic of interest. Experiments with a pooled CRISPR library are significantly more complicated than utilizing CRISPR to change a single genetic region. [[Addgene Link](https://www.addgene.org/crispr/libraries/)]
    - Human Activity-Optimized CRISPR Knockout Library (3 sub-libraries in lentiCRISPRv1) [[Addgene:#1000000100](https://www.addgene.org/pooled-library/sabatini-crispr-human-high-activity-3-sublibraries/) | [DOI Link](https://doi.org/10.1126/science.aac7041) | [PMID: 26472758](https://pubmed.ncbi.nlm.nih.gov/26472758/) | Sabatini/Lander lab]
    - Human CRISPR Knockout Pooled Library (GeCKO v2) [Addgene:#1000000048,#1000000049]()
    - Human CRISPR Metabolic Gene Knockout Library [[Addgene:#110066](https://www.addgene.org/pooled-library/sabatini-human-crispr-metabolic-knockout/) | [DOI Link](https://doi.org/10.1016/j.cell.2015.07.016) | [PMID: 26232224](https://pubmed.ncbi.nlm.nih.gov/26232224/) | Sabatini lab]
    - Drosophila Cell CRISPR Knockout Library [[Addgene:#134582,#134583,#134584](https://www.addgene.org/pooled-library/perrimon-drosophila-crispr-knockout/) | [DOI Link](https://doi.org/10.7554/elife.36333) | [PMID: 30051818](https://pubmed.ncbi.nlm.nih.gov/30051818/) | Norbert Perrimon lab]


### RNA target Cas13 based
> Compared to Cas9-mediated DNA editing, Cas13-based RNA editing tools target dynamically transcribed RNA without permanent alteration of the genome, and the effect of RNA editing can be controlled by means of dose adjustment, etc., which is reversible and quite safer.
---

- RNA knockdown: Cas13d(RfxCas13d or CasRx, Cas13d from Ruminococcus flavefaciens XPD3002) [[DOI Link](https://doi.org/10.1016/j.cell.2018.02.033) | PMID:[29551272](https://pubmed.ncbi.nlm.nih.gov/29551272/)]


- RNA base editor: **REPAIR**(RNA Editing for Programmable A to I Replacement, dCas13-ADAR2): Using catalytically inactive Cas13 (dCas13) to direct adenosine-to-inosine deaminase activity by ADAR2. [[DOI Link](https://doi.org/10.1126/science.aaq0180) | PMID:[ 29070703](https://pubmed.ncbi.nlm.nih.gov/29070703/)]


### DNA base editor



### Prime editor(PE)
  - Split PE system
  - RT families:

### Retron based genome editing





---

## Optogenetics tools

- Neuronal activity activation: 
ChR2(Channelrhodopsin-2)[hChR2(H134R) | ChR2(C128A/C128S)]


- Neuronal activity scilence: NpHR(Halorhodopsin), GtaCR1

- Light-activated Cyclases: PAC & Cyclop



### Reference
1. [Resource website of Optogenetics in neuroscience](https://web.stanford.edu/group/dlab/optogenetics/), Karl Deisseroth lab
2. [Georg Nagel lab, University of Würzburg](https://www.biozentrum.uni-wuerzburg.de/en/bot1/research/prof-dr-georg-nagel/)
3. [Peter Hegemann lab, Humboldt-University of Berlin](https://www.unisyscat.de/people/current-group-leaders/hegemann-peter.html)
4. [Bianxiao Cui Lab, Stanford](https://cuilab.stanford.edu/)
    - light-activatable ERK and AKT signaling pathways
    - light-activatable receptor tyrosine kinases
    - light-activatable molecular motors

## Magnetogenetics tools
> Magnetogenetics is a newly opened field that take advantages of genetically encoded proteins and protein assemblies that are are able to convert magnetic force signal into protein movement and conformation change which may can be applied to manipulate cell behavior.
---

**Magnetic field sensor protein:**
- **MagR(_ISCA1_)**
    - Long, X., Ye, J., Zhao, D. & Zhang, S.-J. Magnetogenetics: remote non-invasive magnetic activation of neuronal activity with a magnetoreceptor. Sci Bull (Beijing) 60, 2107–2119 (2015). [[PMID: 26740890](https://pubmed.ncbi.nlm.nih.gov/26740890/) | [DOI Link](http://dx.doi.org/10.1007/s11434-015-0902-0)]

- **TRPV4-P2A-ferritin**
    - Wheeler, M. A. et al. Genetically targeted magnetic control of the nervous system. Nat Neurosci 19, 756–761 (2016). [[PMID: 26950006](https://pubmed.ncbi.nlm.nih.gov/26950006/) | [DOI Link](https://doi.org/10.1038/nn.4265)]

- **ferritins**: When isolated from cells and loaded with iron in vitro, these crystals generate magnetic forces that are 9 orders of magnitude larger than the forces from the single ferritin cages used in previous studies. 
    - Li, T. L. et al. Engineering a Genetically Encoded Magnetic Protein Crystal. Nano Lett 19, 6955–6963 (2019). [[PMID: 31552740](https://pubmed.ncbi.nlm.nih.gov/31552740/) | [DOI Link](https://doi.org/10.1021/acs.nanolett.9b02266)]
     


## Ref Labs
1. [Sheng-Jia Zhang Lab, Xinqiao Hospital, China](https://www.researchgate.net/scientific-contributions/Sheng-Jia-Zhang-38282418)
2. [Bianxiao Cui Lab, Stanford](https://cuilab.stanford.edu/)
    - Magnetic force-activatable molecular motors

---
## Protein localization motifs
1. **ER localization**
    - **ER-export motifs**: MLLPVPLLLGLLGAAAD (N terminal on insert)
        - FCYENE(Kir2.1 channel[[PMID: 11209084](https://pubmed.ncbi.nlm.nih.gov/11209084/)] | [PMID: 11287009](https://pubmed.ncbi.nlm.nih.gov/11287009/))
    - **ER-targeting sequence**: MLLPVPLLLGLLGAAAD (N terminal on insert)
    - **ER-retention sequence**: KDEL(Lys-Asp-Glu-Leu), HDEI(His-Asp-Glu-Ile) (C terminal on insert)

2. **Mitochondria localization**
    - **Transit peptide**: the signal sequence consisting of a positively charged amino acid (Arg) and an uncharged amino acid (Ser) at the N terminus of the guide peptide of proteins, which enters the mitochondria.
3. **Golgi localization**：


4. **Lysosome localization**：


5. **Cell membrane localization**: 
    - C-terminal CAAX motif of ras proteins [[PMID: 2208277](https://pubmed.ncbi.nlm.nih.gov/2208277/)]

6. **Nucleus**
    - **Nuclear localization signal(NLS)**: Normally included Pro-Lys-Lys-Lys-Lys-Arg-Val.
    - **Nuclear export signal(NES)**: Interphase arrangement of hydrophobic amino acids on ribosomal proteins
8. **Peroxisomal targeting signal(PTS)**: C-terminal SKL i.e. Ser-Lys-Leu


9. **Axon targeting**:



## Prediction of the subcellular localization of proteins
1. [PSORT](http://psort.nibb.ac.jp/)：By amino acid composition information and sorting signal knowledge

2. [TargetP2.0](https://services.healthtech.dtu.dk/service.php?TargetP-2.0)：TargetP-2.0
Subcellular location of proteins: mitochondrial, chloroplastic, secretory pathway, or other
TargetP-2.0 server predicts the presence of N-terminal presequences: signal peptide (SP), mitochondrial transit peptide (mTP), chloroplast transit peptide (cTP) or thylakoid luminal transit peptide (lTP). For the sequences predicted to contain an N-terminal presequence a potential cleavage site is also predicted.
    - **By discriminating the individual targeting signal peptide**

3. [MitoProt](http://ihg.gsf.de/ihg/mitoprot.html)：By discriminating mitochondrial and chloroplast signal peptide

4. [Predotar](http://www.inra.fr/Internet/Produits/Predotar/): By discriminating mitochondrial, chloroplast signal peptide


## Prediction of the subcellular localization of proteins
1. [PSORT](http://psort.nibb.ac.jp/)：By amino acid composition information and sorting signal knowledge

2. [TargetP2.0](https://services.healthtech.dtu.dk/service.php?TargetP-2.0)：TargetP-2.0
Subcellular location of proteins: mitochondrial, chloroplastic, secretory pathway, or other
TargetP-2.0 server predicts the presence of N-terminal presequences: signal peptide (SP), mitochondrial transit peptide (mTP), chloroplast transit peptide (cTP) or thylakoid luminal transit peptide (lTP). For the sequences predicted to contain an N-terminal presequence a potential cleavage site is also predicted.
    - **By discriminating the individual targeting signal peptide**

3. [MitoProt](http://ihg.gsf.de/ihg/mitoprot.html)：By discriminating mitochondrial and chloroplast signal peptide

4. [Predotar](http://www.inra.fr/Internet/Produits/Predotar/): By discriminating mitochondrial, chloroplast signal peptide

5. [NNPSL](http://predict.sanger.ac.uk/nnpsl)：By amino acid composition   

6. [SobLoc](http://www.bioinfo.tsinghua.edu.cn/SubLoc/)：By amino acid composition   
    - **By more sequence information besides the amino acid composition**\
[ ] Nair, Rajesh, and Burkhard Rost. "Better prediction of sub‐cellular localization by combining evolutionary and structural information." Proteins: Structure, Function, and Bioinformatics 53.4 (2003): 917-930. [[DOI Link](https://doi.org/10.1002/prot.10507)]

7. [SignalP 5.0](): SignalP 5.0 is a deep neural network–based method combined with conditional random field that distinguishes between various types of SPs across all domains of life and between three kinds of prokaryotic SPs (Sec/SPI, Sec/SPII, and Tat/SPI).
[PMID: 30778233](https://pubmed.ncbi.nlm.nih.gov/30778233/)

8. [DeepLoc 1.0](https://services.healthtech.dtu.dk/service.php?DeepLoc-1.0): uses a deep learning architecture very similar to what we have used in this study to predict the subcellular localisation of proteins. [[DeepLoc 2.0](https://services.healthtech.dtu.dk/service.php?DeepLoc-2.0)]
- Almagro Armenteros, José Juan, et al. "DeepLoc: prediction of protein subcellular localization using deep learning." Bioinformatics 33.21 (2017): 3387-3395. [[PMID: 29036616](https://pubmed.ncbi.nlm.nih.gov/29036616/) | [DOI Link](https://doi.org/10.1093/bioinformatics/btx431)]


## Open Source & Commercial Softwares
1. [Pymol](https://pymol.org/2/); [PyMOL Wiki](https://pymolwiki.org/index.php/Main_Page)
2. [SnapGene](https://www.snapgene.com/)
3. [Benchling](https://www.benchling.com/)
4. [NEB cloner](https://nebcloner.neb.com/#!/)

## Vendor company
1. [New England Biolabs(NEB)](https://international.neb.com/): NEB is a leader in the discovery and development of molecular biology reagents.
    - Restriction enzymes
    - Gibson Mix [[Manual]() | [Source]()]
2. [Addgene](https://www.addgene.org/): Addgene is a non-profit plasmid repository. Addgene facilitates the exchange of genetic material between laboratories by offering plasmids and their associated cloning data to not-for-profit laboratories around the world.
3. [Vazyme](https://www.vazyme.com/): 
    - One-step cloning kit [[Manual]() | [Source]()]
4. [Geneviz from Azenta](https://www.genewiz.com/): As a leader in R&D genomics services, GENEWIZ provides superior data and high-quality constructs for next generation sequencing, gene synthesis, and sanger sequencing.
    - Primer(Oligos) synthesis
    - Gene(dsDNA) synthesis and subcloning
5. [Takara](https://www.takarabio.com/products/pcr/high-yield-pcr): Takara provide high-quality PCR polymerases.
    - PrimeSTAR HS Polymerase [[Manual](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ESuaz2tU5xVBl5JBOENFbIcBln_RBGmfKGnq9zWmFmQwuQ?e=YdiQod) | [Cat. #: R040A](https://www.takarabio.com/products/pcr/high-fidelity-pcr/primestar-hs-dna-polymerase)]
6. [Thermo Scientific](https://www.thermofisher.com/search/browse/category/us/en/90220140/enzymes+and+inhibitors)
    - T4 DNA ligase [[Manual](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EX57gCIxJENDr2YmEfO7hYQB2uYgXV7BPZlZeZm0h3-EeA?e=g9qahA) | [Cat. #: EL0011](https://www.thermofisher.com/order/catalog/product/EL0011)]

2. [Addgene](https://www.addgene.org/): Addgene is a non-profit plasmid repository. Addgene facilitates the exchange of genetic material between laboratories by offering plasmids and their associated cloning data to not-for-profit laboratories around the world.


## Ref Labs
1. [Feng Zhang Lab, MIT](https://zlab.bio/)
2. [David Liu Lab, Harvard](https://www.liugroup.us/)
3. [Karl Deisseroth Lab, Stanford](https://web.stanford.edu/group/dlab/about_pi.html)
4. [Adam Cohen Lab, Harvard](http://cohenweb.rc.fas.harvard.edu/)
4. [Yulong Li Lab, PKU](http://www.yulonglilab.org/)
5. [Jin Zhang Lab, UCSD](http://jinzhanglab.ucsd.edu/)




---
## Molecular Cloning Basis
1. Performance Chart for Restriction Enzymes [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EcwACbUakrtJiDUsLMcYFwIBuJ4Ou_lzt9Xj-jiVglstbg?e=usbUiE)]
2. NEB biolabs catalog technical references [[PDF](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EeE2HTSyigtPoTB-XQAlQMgByIS8ytW1LVmneBgjgA57Eg?e=w0k38a)]




### Online Courses:
- **Molecular Biology, MIT, EdX**
1. **Part 1: DNA Replication and Repair**: An in-depth adventure through DNA replication and repair to strengthen your scientific thinking and experimental design skills. [[Source](https://www.edx.org/course/molecular-biology-part-2-transcription-and-transposition)]
    
2. **Part 2: Transcription and Transposition**: Strengthen your scientific thinking and experimental design skills in this adventure through transcription and transposition. [[Source](https://www.edx.org/course/molecular-biology-part-2-transcription-and-transposition)]
    - Week 1 : Machinery and Promoters of Bacterial Transcription
    
3. **Part 3: RNA Processing and Translation**: An in-depth adventure through RNA Processing and Translation. Strengthen your scientific thinking and biological experimental design skills. [[Source](https://www.edx.org/course/molecular-biology-part-3-rna-processing-and-transl)]




---
## Nobel laureates
- **Emmanuelle Charpentier**, Max Planck Unit for the Science of Pathogens, Berlin, Germany\
The Nobel Prize in Chemistry 2020\
**For the development of a method for genome editing** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EdFqVEkJF0VIrA7BvepNFJIBf72T_osrFneQ4oBF3krq3w?e=Zg1j5C)]

- **Jennifer A. Doudna**, University of California, Berkeley, CA, USA\
The Nobel Prize in Chemistry 2020\
**The Chemistry of CRISPR: Editing the Code of Life** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EdFqVEkJF0VIrA7BvepNFJIBf72T_osrFneQ4oBF3krq3w?e=Zg1j5C)]

- **Osamu Shimomura**, Marine Biological Laboratory (MBL), Woods Hole, MA, USA; Boston University Medical School, Massachusetts, MA, USA\
The Nobel Prize in Chemistry 2008\
**Discovery of Green Fluorescent Protein, GFP** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/Ea7xbY7WDeFMjWMlFhE5Rr8BBd-j396_n-03NDA4n3CnsA?e=Qz2IXt) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EU8OcGx8TRhEn-haQRwSFb4BauHuAkiGBI2PjtxSXr9XKg?e=4mwPl8) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERYnt5iVZXlPmIs7cG8Z8DIB9h4nRGs9GM_vI6x-hiJgDQ?e=6FHFNa) |  [Source](https://www.nobelprize.org/prizes/chemistry/2008/shimomura/lecture/)]

- **Martin Chalfie**, Columbia University, New York, NY, USA\
The Nobel Prize in Chemistry 2008\
**GFP: Lighting Up Life** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EQFIlZcij4dHt1ziRU-APFsBGLj_jXt8Ehvz1gKYnyWw9g?e=d8d8x4) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EUB6WdVEbLNCs1laNYyrE1ABL5QxOSOO3wlGXeAq81uKwg?e=ZQRTaZ) | [Source](https://www.nobelprize.org/prizes/chemistry/2008/chalfie/lecture/)]

- **Roger Y. Tsien**, University of California, San Diego, CA, USA\
The Nobel Prize in Chemistry 2008\
**Constructing and Exploiting the Fluorescent Protein Paintbox** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EX_KX-KLPsFBhgk9pOovOzIBqp8mjNiJMFrchuwG72LKxg?e=9C9yCR) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EbXNqpxXkjpIrhgjOkKCvzAB_K3sN2tHir1mLW5t61ea4w?e=q1npQ2) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EVZYQT1jreBBrSaWJ-MMmU8Bja7PbbZo16NCLwFhcSoJRg?e=QwVmgh) |  [Source](https://www.nobelprize.org/prizes/chemistry/2008/tsien/facts/)]

- **Craig C. Mello**, University of Massachusetts Medical School, Worcester, MA, USA\
The Nobel Prize in Physiology or Medicine 2006\
**Return to the RNAi World: Rethinking Gene Expression and Evolution** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/ERJCKx_Rxm5DqAj_FwjVRQoB514JTZqibAK96j2bjrkALg?e=z9fTmu) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Eao2Yc7A65xEkWg53nQmKd8B-cRBCjfUSdwCn0yOD2cEZg?e=JcBHKl) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EXIuNu-X9aFBmESlD6lhERcBVK6IZ-6J-OKB5-nAM0OJfQ?e=ekQ3KO) |  [Source](https://www.nobelprize.org/prizes/medicine/2006/mello/facts/)]

- **Andrew Z. Fire**, Stanford University School of Medicine, Stanford, CA, USA\
The Nobel Prize in Physiology or Medicine 2006\
**Gene Silencing by Double Stranded RNA** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/ERJCKx_Rxm5DqAj_FwjVRQoB514JTZqibAK96j2bjrkALg?e=z9fTmu) | [Lecture Slides](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Eao2Yc7A65xEkWg53nQmKd8B-cRBCjfUSdwCn0yOD2cEZg?e=JcBHKl) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EXIuNu-X9aFBmESlD6lhERcBVK6IZ-6J-OKB5-nAM0OJfQ?e=ekQ3KO) |  [Source](https://www.nobelprize.org/prizes/medicine/2006/fire/facts/)]

- **Phillip A. Sharp**, Massachusetts Institute of Technology (MIT), Center for Cancer Research, Cambridge, MA, USA\
The Nobel Prize in Physiology or Medicine 1993\
**Split Genes and RNA Splicing** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EdFqVEkJF0VIrA7BvepNFJIBf72T_osrFneQ4oBF3krq3w?e=Zg1j5C) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EXAic-FpAitMhZO63Hu_Fk0BLgmFdBfA5ekBdp-M6YXerA?e=oXw1fI) | [Source](https://www.nobelprize.org/prizes/medicine/1993/sharp/facts/)]

- **Richard Roberts**, New England Biolabs, Beverly, MA, USA\
The Nobel Prize in Physiology or Medicine 1993\
**An Amazing Distortion in DNA Induced by a Methyltransferase** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EffVy-YZa5xJhCho-d_ClsQBpUu5KI8463FRFW6BZ8t1Dw?e=9I2cS2) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EURYg9C5uWJDhxl5SriTYugB6RKFveeKkXnfq9xOAeYVHw?e=SFvfnq) | [Source](https://www.nobelprize.org/prizes/medicine/1993/roberts/facts/)]

- **Kary B. Mullis**, Cetus Corporation\
The Nobel Prize in Chemistry 1993\
**The Polymerase Chain Reaction** [[Read the Lecture](https://www.nobelprize.org/prizes/chemistry/1993/mullis/lecture/) | [Source](https://www.nobelprize.org/prizes/chemistry/1993/mullis/facts/)]

- **Michael Smith**, University of British Columbia, Vancouver, Canada\
The Nobel Prize in Chemistry 1993\
**Synthetic DNA and Biology** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EQxGZFMgJhhDoVnAPN8Bro4BIM8CP1UgV9XUuyeiK0CVQQ?e=273ecb) | [Source](https://www.nobelprize.org/prizes/chemistry/1993/smith/facts/)]

- **Susumu Tonegawa**, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA\
The Nobel Prize in Physiology or Medicine 1987\
**Somatic Generation of Immune Diversity** [[Nobel Lecture video](https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EXrmZU1m_VxLtWWavzDn3s0Bw7013B1NEqK7RZ2AOrCwEQ?e=D6e6Az) | [Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERFkAs1pvepClNLYtNEEPDQBBrMX_t-wbkoKzRH0NRkUxw?e=QWs5gr) | [Source](https://www.nobelprize.org/prizes/medicine/1987/tonegawa/lecture/)]

- **Paul Berg**, Stanford University, Stanford, CA, USA\
The Nobel Prize in Chemistry 1980\
**Dissections and Reconstructions of Genes and Chromosomes** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Eei88f7LZ4hOrkvR5Irn2QABVfOyABK3paWRENRv3i9ZKg?e=kwt7Fv) | [Source](https://www.nobelprize.org/prizes/chemistry/1980/berg/lecture/)]

- **Daniel Nathans**, Johns Hopkins University School of Medicine, Baltimore, MD, USA\
The Nobel Prize in Physiology or Medicine 1978\
**Restriction Endonucleases, Simian Virus 40, and the New Genetics** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EXU0uVvUd6JEr9Z2CyCwtXoBKfAgsNO2Tg78BAbFvOpLrg?e=MdCExv) | [Source](https://www.nobelprize.org/prizes/medicine/1978/nathans/facts/)]

- **Werner Arber**, Biozentrum der Universität, Basel, Switzerland\
The Nobel Prize in Physiology or Medicine 1978\
**Promotion and Limitation of Genetic Exchange** [[Read the Lecture](https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/Eei88f7LZ4hOrkvR5Irn2QABVfOyABK3paWRENRv3i9ZKg?e=tHafsw) | [Source](https://www.nobelprize.org/prizes/medicine/1978/arber/facts/)] 

- **Hamilton O. Smith**, Johns Hopkins University School of Medicine, Baltimore, MD, USA\
The Nobel Prize in Physiology or Medicine 1978\
**Nucleotide Sequence Specificity of Restriction Endonucleases** [[Read the Lecture]() | [Source](https://www.nobelprize.org/prizes/medicine/1978/smith/facts/)]

