<!DOCTYPE html>
<html lang="en-us">
  <head>
    
    <script type="application/ld+json">

{  
  "@context":"http://schema.org",
  "@type":"Website",
  "@id":"https:\/\/leoj.xyz\/",
  "author": {
    "@type": "Person",
    "name": "Jun-Liszt Li",
    
    "image": "https://leoj.xyz/img/Dingtalk_20220815163324.jpg"
    
  },
  "name":"Jun-Liszt Li",
  "description":"\u003cblockquote\u003e\n\u003cp\u003eMy own prejudices are exactly the opposite of the functionalists\u0026rsquo;: \u0026ldquo;If you want to understand function, study structure\u0026rdquo;.\u003cbr\u003e\n\u0026mdash; Francis Crick\u003c\/p\u003e\n\u003c\/blockquote\u003e",
  "url":"https:\/\/leoj.xyz\/resource\/structure_bio\/",
  "keywords":"[tech]"
}

</script>
    <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="generator" content="Hugo 0.126.3 with theme Tranquilpeak 0.5.3-BETA">
<meta name="author" content="Jun-Liszt Li">
<meta name="keywords" content="tech">
<meta name="description" content="
My own prejudices are exactly the opposite of the functionalists&rsquo;: &ldquo;If you want to understand function, study structure&rdquo;.
&mdash; Francis Crick
">


<meta property="og:description" content="
My own prejudices are exactly the opposite of the functionalists&rsquo;: &ldquo;If you want to understand function, study structure&rdquo;.
&mdash; Francis Crick
">
<meta property="og:type" content="article">
<meta property="og:title" content="Structure Biology">
<meta name="twitter:title" content="Structure Biology">
<meta property="og:url" content="https://leoj.xyz/resource/structure_bio/">
<meta property="twitter:url" content="https://leoj.xyz/resource/structure_bio/">
<meta property="og:site_name" content="Jun-Liszt Li">
<meta property="og:description" content="
My own prejudices are exactly the opposite of the functionalists&rsquo;: &ldquo;If you want to understand function, study structure&rdquo;.
&mdash; Francis Crick
">
<meta name="twitter:description" content="
My own prejudices are exactly the opposite of the functionalists&rsquo;: &ldquo;If you want to understand function, study structure&rdquo;.
&mdash; Francis Crick
">
<meta property="og:locale" content="en-us">

  
    <meta property="article:published_time" content="2022-08-21T13:24:59">
  
  
    <meta property="article:modified_time" content="2022-08-21T13:24:59">
  
  
  
    
      <meta property="article:section" content="category">
    
      <meta property="article:section" content="subcategory">
    
  
  
    
      <meta property="article:tag" content="CryoEM">
    
      <meta property="article:tag" content="X-Ray">
    
      <meta property="article:tag" content="Protein purification">
    
  


<meta name="twitter:card" content="summary">







  <meta property="og:image" content="https://leoj.xyz/img/Dingtalk_20220815163324.jpg">
  <meta property="twitter:image" content="https://leoj.xyz/img/Dingtalk_20220815163324.jpg">





  <meta property="og:image" content="https://leoj.xyz/img/7uhy_GATOR2.jpeg">
  <meta property="twitter:image" content="https://leoj.xyz/img/7uhy_GATOR2.jpeg">


    <title>Structure Biology</title>

    <link rel="icon" href="https://leoj.xyz/favicon.png">
    

    

    <link rel="canonical" href="https://leoj.xyz/resource/structure_bio/">

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha512-H9jrZiiopUdsLpg94A333EfumgUBpO9MdbxStdeITo+KEIMaNfHNvwyjjDJb+ERPaRS6DpyRlKbvPUasNItRyw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    
    
    
    <link rel="stylesheet" href="https://leoj.xyz/css/style-h6ccsoet3mzkbb0wngshlfbaweimexgqcxj0h5hu4h82olsdzz6wmqdkajm.min.css" />
    
    

    
      
  
    
      <script async src="https://www.googletagmanager.com/gtag/js?id=327157081"></script>
      <script>
        var doNotTrack = false;
        if ( false ) {
          var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
          var doNotTrack = (dnt == "1" || dnt == "yes");
        }
        if (!doNotTrack) {
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());
          gtag('config', '327157081');
        }
      </script>
    
  


    
    
  </head>

  <body>
    <div id="blog">
      <header id="header" data-behavior="4">
  <i id="btn-open-sidebar" class="fa fa-lg fa-bars"></i>
  <div class="header-title">
    <a class="header-title-link" href="https://leoj.xyz/" aria-label="Go to homepage">Jun-Liszt Li</a>
  </div>
  
    
      <a class="header-right-picture "
         href="https://leoj.xyz/#about" aria-label="Open the link: /#about">
    
    
    
      
        <img class="header-picture" src="https://leoj.xyz/img/Dingtalk_20220815163324.jpg" alt="Author&#39;s picture" />
      
    
    </a>
  
</header>

      <nav id="sidebar" data-behavior="4">
  <div class="sidebar-container">
    
      <div class="sidebar-profile">
        <a href="https://leoj.xyz/#about" aria-label="Read more about the author">
          <img class="sidebar-profile-picture" src="https://leoj.xyz/img/Dingtalk_20220815163324.jpg" alt="Author&#39;s picture" />
        </a>
        <h4 class="sidebar-profile-name">Jun-Liszt Li</h4>
        
          <h5 class="sidebar-profile-bio">Hello, My name is Jun-Liszt Li (Leo). my scientific interest is to understand the molecular logic of cell growth and develop new tools to address specific questions in this path.</h5>
        
      </div>
    
    <ul class="sidebar-buttons">
      
  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/page" title="Home">
    
      <i class="sidebar-button-icon fas fa-lg fa-home" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Home</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/researchs" title="Research">
    
      <i class="sidebar-button-icon fas fa-lg fa-dna" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Research</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/page/publication" title="Publication">
    
      <i class="sidebar-button-icon fas fa-lg fa-scroll" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Publication</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/posts" title="Blog">
    
      <i class="sidebar-button-icon fas fa-lg fa-blog" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Blog</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/tooldevelopment" title="Tool develop">
    
      <i class="sidebar-button-icon fas fa-lg fa-wrench" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Tool develop</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/resource" title="Resource">
    
      <i class="sidebar-button-icon fas fa-lg fa-warehouse" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Resource</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/page/photos" title="Photos">
    
      <i class="sidebar-button-icon fas fa-lg fa-camera" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Photos</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/page/about" title="About">
    
      <i class="sidebar-button-icon fas fa-lg fa-question" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">About</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/page/contact" title="Contact">
    
      <i class="sidebar-button-icon fas fa-lg fa-at" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Contact</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/categories" title="Categories">
    
      <i class="sidebar-button-icon fas fa-lg fa-bookmark" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Categories</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/tags" title="Tags">
    
      <i class="sidebar-button-icon fas fa-lg fa-tags" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Tags</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/archives" title="Archives">
    
      <i class="sidebar-button-icon fas fa-lg fa-archive" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Archives</span>
    </a>
  </li>


    </ul>
    <ul class="sidebar-buttons">
      

    </ul>
    <ul class="sidebar-buttons">
      
  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://github.com/Jun-Lizst" target="_blank" rel="noopener" title="GitHub">
    
      <i class="sidebar-button-icon fab fa-lg fa-github" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">GitHub</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://scholar.google.com/citations?user=AGPeLy0AAAAJ&amp;hl=en" target="_blank" rel="noopener" title="Google Scholar">
    
      <i class="sidebar-button-icon fab fa-lg fa-google" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Google Scholar</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://twitter.com/poppingjun" target="_blank" rel="noopener" title="Twitter">
    
      <i class="sidebar-button-icon fab fa-lg fa-twitter" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Twitter</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://www.linkedin.com/in/jun-liszt-li-ph-d-32916aa3" target="_blank" rel="noopener" title="Linkedin">
    
      <i class="sidebar-button-icon fab fa-lg fa-linkedin" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">Linkedin</span>
    </a>
  </li>

  <li class="sidebar-button">
    
      <a class="sidebar-button-link " href="https://leoj.xyz/index.xml" title="RSS">
    
      <i class="sidebar-button-icon fas fa-lg fa-rss" aria-hidden="true"></i>
      
      <span class="sidebar-button-desc">RSS</span>
    </a>
  </li>


    </ul>
  </div>
</nav>

      

      <div id="main" data-behavior="4"
        class="
               hasCoverMetaIn
               ">
        <article class="post" id="top">
          
          
            <div class="post-header main-content-wrap text-left">
  
    <h1 class="post-title">
      Structure Biology
    </h1>
  
  
</div>
          
          <div class="post-content markdown">
            <div class="main-content-wrap">
              <blockquote>
<p>My own prejudices are exactly the opposite of the functionalists&rsquo;: &ldquo;If you want to understand function, study structure&rdquo;.<br>
&mdash; Francis Crick</p>
</blockquote>
<hr>
<h1 id="cryoem-single-particle-analysis-spa">CryoEM Single Particle Analysis (SPA)</h1>
<h3 id="-key-hardwares"><strong>| Key Hardwares</strong></h3>
<ol>
<li>
<p><strong>Cryo-EM microscopy</strong></p>
<ul>
<li><a href="https://www.thermofisher.com/cz/en/home/electron-microscopy/products/transmission-electron-microscopes/krios-cryo-tem/features.html">300 kV Titan Krios</a>: The Titan <strong>Krios G4</strong> from Thermo Fisher Scientific is a 300 kV transmission electron microscope (TEM) with a three-condenser lens system. It features the latest technology for high-end single-particle and tomography data acquisition including a cold field emission gun (C-FEG), SelectrisX energy filter, Falcon 4 electron counting camera and volta phase plate.</li>
<li><a href="">200 kV</a>: The cryo-grids were initially screened at a nominal magnification of ×92,000 in an 200 kV microscope.
<ul>
<li><strong>Glacios</strong>: The Glacios from Thermo Fisher Scientific is a 200 kV transmission electron microscope with a two-condenser lens system.</li>
<li><strong>Talos Arctica</strong>: FEI <a href="https://documents.thermofisher.com/TFS-Assets/MSD/Datasheets/DS0189-EN-11-2019-Thermo-Scientific-Talos-Arctica-WEB.pdf">Talos Arctica microscope (200 kV)</a>, equipped with an FEI Ceta camera. [<a href="https://bioem.shanghaitech.edu.cn/_upload/article/files/79/11/d357319b47e584a520a8bbd06351/b83e470e-0a1b-4f87-a897-ea30e42efe93.pdf">Talos L120C manual, PDF</a> | <a href="https://bioem.shanghaitech.edu.cn/_upload/article/videos/63/19/77aa69a04eca8fbc00f84f182337/b0817747-7020-4f37-80a3-d94cd4ed4365-B.mp4">Tutorial video</a> | <a href="https://emcore.ucsf.edu/content/instructions-arctica-screening">Instruction for Arctica screening, UCSF</a>]</li>
</ul>
</li>
<li><a href="">40kV-120kV</a>: <a href="https://emcore.ucsf.edu/fei-t12">FEI Tecnai T12</a> is mainly used for simple screening and data collection of negatively-stained samples. It&rsquo;s also used for cryo-EM training and screening of vitrified grids.
<ul>
<li>FEI T12 (FEI Tecnai T12 Transmission Electron Microscope) operates at 120kV with a LaB6 filament and is equipped with a Gatan UltraScan 895 4k CCD.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>Direct Electron Detectors (DEDs)</strong>:
The advent of Direct Electron Detectors (DEDs) marks the definitive &ldquo;singularity&rdquo; in the history of Cryo-Electron Microscopy (Cryo-EM). Prior to 2013, the field was bottlenecked by traditional Charge-Coupled Devices (CCDs), which suffered from significant signal degradation due to the indirect conversion of electrons to photons.The breakthrough came with DED technology, which allows for the direct sensing of electrons with unprecedented detective quantum efficiency (DQE). By introducing ultra-fast frame rates (Movie Mode), DEDs enabled researchers to overcome the two primary obstacles to high resolution: radiation damage and beam-induced motion. This shift facilitated the &ldquo;Resolution Revolution,&rdquo; transforming Cryo-EM from a &ldquo;blob-ology&rdquo; niche into a premier tool for resolving complex membrane assemblies—such as the spliceosome and gamma-secretase—at near-atomic scales.</p>
</li>
</ol>
<table>
<thead>
<tr>
<th>Detector</th>
<th>Sensor size</th>
<th>Pixel size (um)</th>
<th>Readout speed (fps)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DE-20</td>
<td>5120x3480</td>
<td>6.4</td>
<td>25</td>
</tr>
<tr>
<td>Falcon-II</td>
<td>4096x4096</td>
<td>14.0</td>
<td>18</td>
</tr>
<tr>
<td>K2 Summit</td>
<td>3838x3710</td>
<td>5</td>
<td>400</td>
</tr>
</tbody>
</table>
<h3 id="-sample-preparation"><strong>| Sample preparation</strong></h3>
<ol>
<li>
<p><strong>Methods to improve particle behavior</strong></p>
<ul>
<li><strong>Size exclusion chromatography</strong>: Gel filtration can be a powerful tool to predict structural heterogeneity. Whenever possible, gel filtration should be incorporated to the final steps of a purification protocol.</li>
<li><strong>Protein or protein complex stabilization</strong>: If negative stain or cryo-EM analysis reveals significant structural heterogeneity, then the user may consider additional methods to stabilize a structural target in a defined conformation. These approaches have proven to be effective in trapping conformationally variable proteins and complexes in energetic minima that allow high-resolution structure determination. Examples include using small molecule inhibitors, non-hydrolyzable nucleotides (if applicable), Fab antibody fragments, or genetic perturbations (e.g., mutating catalytic residues).
<ul>
<li>[<a href="">ALFA tag (SRLEEELRRRLTE) and ALFA nanobody</a> | <a href="https://www.nature.com/articles/s41467-023-42210-9#Sec2">Reference</a>]</li>
</ul>
</li>
<li><strong>Chemical crosslinking</strong>: One common way to counteract structural heterogeneity is to chemically crosslink the purified sample, such as with glutaraldehyde or BS3. However, crosslinked samples run a risk of producing structural artifacts and this approach should therefore be considered only after a control, non-crosslinked sample has been evaluated.</li>
<li><strong>Membrane proteins</strong>:
<ul>
<li><strong>Isolation of membrane fractions</strong>: 100,000 g</li>
<li><strong>Screen of Detergents</strong>: [CHAPS | Digitonin | 1% DDM + 0.2% CHS | 2% (w/v) GDN + 0.1% (w/v) CHS + 0.2 mg/ml Avanti | 0.5% LMNG + 0.03% CHS | AMPHIPOL A8-35 | AMPHIPOL PMAL-C8]</li>
<li><strong>Affinity Chromatography (AC)</strong>: [anti-FLAG | Strep-Tactin XT]</li>
<li><strong>Crosslinking for membrane protein complex</strong>: 0.1% (v/v) , 40 mins</li>
<li><strong>SEC</strong>: [[Superose 6 increase 10/300] | [Superdex 200 increase 10/300]]</li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>Protocol for grid preparation</strong></p>
<ul>
<li>
<p>For Negative stain TEM: [<a href="https://cryoem.wisc.edu/wp-content/uploads/sites/341/2021/01/Negative_Stain_Grid_Preparation_21Jan2021.pdf">Protocol, PDF</a> | <a href="https://www.ncbi.nlm.nih.gov/pubmed/22215030">Yifan Cheng lab, 2011</a> | <a href="https://emcore.ucsf.edu/content/negative-stain-protocol">UCSF protocol</a>]
&mdash; General procedure: 3.5 μL of purified protein complexes at 0.02 mg/ml was pipetted onto a continuous carbon coated grid and incubated at room temperature for 60 s. The sample was blotted using filter paper. Immediately, 3.5 μL of 2% (w/v) uranyl formate was pipetted onto the grid and blotted away with filter paper. This treatment with uranyl formate stain was repeated two more times, except an addition of a 30 s wait between stain application and blotting during the last cycle. The grid was left to dry at room temperature for 2 mins.
&mdash; Cressington 208 carbon evaporator [<a href="https://emcore.ucsf.edu/content/cressington-208-carbon-evaporator">Carbon coating grids (UCSF manual)</a>]</p>
</li>
<li>
<p>Graphene type <br>
&mdash; <a href="https://www.biorxiv.org/node/981810.full">Monolayer graphene</a> <br>
&mdash; <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119484/">Graphene oxide</a> <br>
&mdash; <a href="https://www.pnas.org/content/116/24/11718">Functionalized graphene</a> <br>
&mdash; <a href="https://www.biorxiv.org/content/10.1101/657411v1.full">Functionalized graphene oxide</a></p>
</li>
<li>
<p>Grids:
&mdash; Quantifoil R 1.2/1.3 300 Holey Carbon films Au 300 mesh grids
&mdash; holey carbon grid (Quantifoil R1.2/1.3)</p>
</li>
<li>
<p>Plasma cleaner for glow discharge
&mdash; Glow Discharge Plasma Cleaner (<a href="https://www.mitegen.com/product/basic-plasma-cleaner/">SKU: M-PDC-32G</a>)
&mdash; Economy Dry Oxygen Service Pump (<a href="https://www.mitegen.com/product/economy-dry-oxygen-service-pump/">SKU: M-PDC-OPE</a>)
&mdash; Vacuum Gauge and Digital Meter (<a href="https://www.mitegen.com/product/vacuum-gauge-digital-meter/">SKU: M-PDC-VCG</a>)
&mdash; Quartz Sample Tray (<a href="https://www.mitegen.com/product/quartz-sample-tray/">SKU: M-PDC-32T</a>)
&mdash; GridPrepBlock Support For Grids During Surface Treatment (<a href="https://www.mitegen.com/product/grid-prep-block/">SKU: M-CEM-GSB5-1</a>)</p>
</li>
</ul>
</li>
<li>
<p><strong>Sample Vitrification</strong>: Vitrification is the rapid cooling of a liquid into a solid, glass-like amorphous state, preventing the formation of damaging ice crystals.</p>
<ul>
<li>
<p><strong>Vitrobot, Thermo Fisher</strong>: The Vitrobot Mark IV System from Thermo Fisher Scientific is an automated plunge freezer able to vitrify thin samples applied to an electron microscopy grid after removing excess fluid by blotting. [<a href="https://bioem.shanghaitech.edu.cn/_upload/article/videos/a4/8e/d17c5c5e4d858b49938c682dca00/b9546e76-893f-4b41-bc9d-d3b4b6244c48-B.mp4">Tutorial video, mp4</a> | <a href="https://bioem.shanghaitech.edu.cn/_upload/article/files/79/11/d357319b47e584a520a8bbd06351/0ac79af7-f4bc-45ff-9fcf-0cf3f357e656.pdf">Manual, PDF</a> | <a href="https://info.cpos.hku.hk/wp-content/uploads/2025/04/Standard-Operation-Protocol-%E2%80%93-Vitrobot-Mark-IV.pdf">Standard Operation Protocol, PDF</a>]</p>
</li>
<li>
<p><a href="https://www.leica-microsystems.com/products/sample-preparation-for-electron-microscopy/p/leica-em-gp2/"><strong>EM GP2, Leica</strong></a>: The EM GP2 from Leica Microsystems is an automated plunge freezer able to vitrify thin samples applied to an electron microscopy grid after removing excess fluid by blotting. [<a href="https://bioem.shanghaitech.edu.cn/_upload/article/videos/d8/77/91cf34504384919c5e5787c54f07/666c8301-1b09-43d0-9472-1d00f5dbe414-B.mp4">Tutorial video, mp4</a> | <a href="https://bioem.shanghaitech.edu.cn/_upload/article/files/79/11/d357319b47e584a520a8bbd06351/883df0db-6349-4fca-9686-a4a8107152e6.pdf">Manual, PDF</a>]</p>
</li>
<li>
<p>Vitrodrop:</p>
</li>
<li>
<p>Preassis</p>
</li>
<li>
<p>CryoSol Vitrojet System</p>
</li>
<li>
<p>Products for improving sample vitrification</p>
<table>
<thead>
<tr>
<th>Product name</th>
<th>Catalog</th>
<th>Vendor</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fluorinated FC-8</td>
<td><a href="https://anatrace.com/en/product/F300F-1-GM">F300F</a></td>
<td>Anatrace</td>
<td>FC-8 has been  vitrification condition for both soluble and membrane proteins</td>
</tr>
<tr>
<td>Fluorinated OM</td>
<td><a href="https://www.atcc.org/products/crl-2266">0310F</a></td>
<td>Anatrace</td>
<td></td>
</tr>
<tr>
<td>Calixar C2B kit</td>
<td><a href="https://www.atcc.org/products/crl-2266">MD1-109</a></td>
<td>MD</td>
<td></td>
</tr>
<tr>
<td>Cryo-EM V-Kit</td>
<td><a href="https://www.atcc.org/products/htb-96">X-CEM-301</a></td>
<td>Jena</td>
<td></td>
</tr>
<tr>
<td>Cryo-EM Vitrification Starter Kit</td>
<td><a href="https://www.mitegen.com/product/cryo-em-vitrification-starter-kit/">M-CEM-VK301</a></td>
<td>MiTeGen</td>
<td>The Cryo-EM V-Kit is a Vitrification Starter Kit that offers both a selection of surfactants which have been successfully applied in cryo-EM sample preparation and a selection of Quantifoil Holey Carbon Films to facilitate the search for the optimal vitrification condition for both soluble and membrane proteins.</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ol>
<h3 id="-cryo-em-data-analysis"><strong>| Cryo-EM data analysis</strong></h3>
<h4 id="softwares">Softwares</h4>
<ol>
<li>
<p><a href="https://guide.cryosparc.com/">CryoSPARC</a>: CryoSPARC (Cryo-EM Single Particle Ab-Initio Reconstruction and Classification) is a state of the art data analysis solution for single-particle analysis (SPA) in cryo-electron microscopy (cryo-EM). CryoSPARC is used to reconstruct and visualize cryo-EM structures of biological targets including membrane proteins, viruses and complexes.</p>
</li>
<li>
<p><a href="https://relion.readthedocs.io/en/release-5.0/">Relion</a>: relion (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination. It is developed in the group of <a href="https://www2.mrc-lmb.cam.ac.uk/groups/scheres/impact.html">Sjors Scheres</a> at the <a href="https://www2.mrc-lmb.cam.ac.uk/group-leaders/n-to-s/sjors-scheres/">MRC Laboratory of Molecular Biology</a>.</p>
</li>
<li>
<p><a href="https://github.com/spider-em/SPIDER">SPIDER</a>: SPIDER (System for Processing Image Data from Electron microscopy and Related fields) is an image processing system for electron microscopy, especially usefull for single-particle reconstruction.</p>
<ul>
<li><a href="https://spider-em.github.io/SPIDER/">Online tutorial</a></li>
<li>Evaluation of difference maps: It can be achieved by Spider and the <a href="https://grigoriefflab.umassmed.edu/diffmap">diffmap program</a> to calculate a difference map between two input maps and the estimated standard deviation of the difference map.</li>
</ul>
</li>
<li>
<p><a href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/">COOT</a>: For manually rebuilt and/or adjust models.</p>
<ul>
<li>Emsley, Paul, and Kevin Cowtan. &ldquo;Coot: model-building tools for molecular graphics.&rdquo; Biological crystallography 60.12 (2004): 2126-2132. [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/">PMID: 15572765</a>]</li>
<li>Using Coot for model building into EM maps: A tutorial [<a href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/files/EM-Tutorial-Coot-PE.pdf">PDF</a>]</li>
</ul>
</li>
<li>
<p><a href="https://phenix-online.org/">Phenix</a>: For Real space and Rosetta refinements.</p>
<ul>
<li>Adams, Paul D., et al. &ldquo;PHENIX: a comprehensive Python-based system for macromolecular structure solution.&rdquo; Biological crystallography 66.2 (2010): 213-221. [<a href="https://pubmed.ncbi.nlm.nih.gov/20124702/">PMID: 20124702</a>]</li>
</ul>
</li>
<li>
<p><a href="https://www.rosettacommons.org/software/">Rosetta Commons</a>: For decades, Rosetta has been at the forefront of computational biology, offering groundbreaking capabilities in the modeling, design and analysis of protein structures.</p>
</li>
<li>
<p><a href="https://www.ccpem.ac.uk/">CCP-EM</a>: Supported by the UK Medical Research Council, the Collaborative Computational Project for Electron cryo-Microscopy (cryoEM) provides support in computational areas for users and developers in biological cryoEM.</p>
<ul>
<li>Single Particle Analysis using Relion [<a href="https://www.ccpem.ac.uk/downloads/tutorials/doppio/Doppio_Relion_SPA_tutorial.pdf">PDF</a>]</li>
<li>Model Building and Validation in CCP-EM Doppio [<a href="https://www.ccpem.ac.uk/downloads/tutorials/doppio/Doppio_model_docking_refinement_tutorial.pdf">PDF</a>]</li>
<li>Refinement against cryo-EM data with CCP-EM using REFMAC &amp; Coot [<a href="https://www.ccpem.ac.uk/downloads/tutorials/refmac_servalcat/servalcat-refmac_coot_tutorial_ECM_2022.pdf">PDF</a>]</li>
</ul>
</li>
<li>
<p><a href="https://guide.cryosparc.com/setup-configuration-and-management/hardware-and-system-requirements">Requirement of computational cluster/Workstation</a>:</p>
<ul>
<li><strong>CPU:</strong> 32 Cores (base clock 2.8GHz+), e.g, AMD Threadripper 3975X</li>
<li><strong>RAM:</strong> 256GB DDR4 @ 3200MHz</li>
<li><strong>Storage:</strong> 4TB PCIe SSD (cache); 200TB RAID 6 storage server via 10Gbps link (raw movies)</li>
<li><strong>GPU:</strong> 4x NVIDIA Quadro GV100, or 4x NVIDIA Tesla V100 or 4x NVIDIA RTX 8000</li>
</ul>
</li>
</ol>
<h4 id="cryoem-data-analysis-protocols">CryoEM data analysis protocols</h4>
<h4 id="deep-learning-tools">Deep learning tools</h4>
<p><strong>Ref softwares</strong>:</p>
<ol>
<li>
<p><strong>MotionCorr</strong>: This program corrects whole frame image motion recordded with dose fractionated image stack. <a href="https://emcore.ucsf.edu/ucsf-motioncor2">MotionCor2</a></p>
<ul>
<li>Xueming Li, Paul Mooney, Shawn Zheng, Chris Booth, Michael B. Braunfeld, Sander Gubbens, David A. Agard and Yifan Cheng (2013) Electron counting and beam-induced motion correction enables near atomic resolution single particle cryoEM. Nature Methods, 10, 584-590. PMID: 23644547</li>
<li><a href="https://github.com/czimaginginstitute/MotionCor3">MotionCor3</a>, an improved implementation of MotionCor2 with addition of CTF (Contrast Transfer Function) estimation, is a multi-GPU accelerated software package that enables single-pixel level correction of anisotropic beam induced sample motion for cryo-electron microscopy and cryo-electron tomography images.</li>
</ul>
</li>
<li>
<p><strong>GeFREALIGN</strong>: FREALIGN is a program developed by Niko Grigorieff laboratory for high-resolution refinement of 3D reconstruction from cryoEM of single particles.</p>
<ul>
<li>Xueming Li, Nikolaus Grigorieff and Yifan Cheng (2010) GPU-enabled FREALIGN: Accelerating single particle 3D reconstruction and refinement in Fourier space on graphics processors. Journal of Structural Biology, advanced publication online June 14 2010. PMID: 20558298</li>
</ul>
</li>
</ol>
<h4 id="online-resources">Online resources</h4>
<ol>
<li>
<p><a href="https://cryoem101.org/"><strong>CryoEM 101</strong></a></p>
</li>
<li>
<p><a href="https://www.thermofisher.com/cz/en/home/electron-microscopy/life-sciences/learning-center/cryo-em-university.html">Cryo-EM University</a>: Cryo-EM University features over 70 hours of theoretical lectures and videos. Created in collaboration with online education expert Professor Grant Jensen, it serves as an introduction to the field and is intended for participants of all levels. Upon completion, you will have a fundamental knowledge of cryo-EM, tips and tricks to overcome sample preparation challenges, and valuable practical advice on the cryo-EM workflow.</p>
</li>
<li>
<p><a href="https://www2.mrc-lmb.cam.ac.uk/research/scientific-training/electron-microscopy/">Electron Cryo-microscopy Course 2023, MRC-LMB</a></p>
<ul>
<li>Specimen preparation for cryo-EM – Katerina Naydenova [<a href="https://youtu.be/hF17d6XjV98">Video</a>]</li>
<li>Cryo-EM Data Collection – Giuseppe Cannone [<a href="https://youtu.be/hS-HrgWv0HQ">Video</a>]</li>
<li>Modelling using Cryo-EM Data: A Review of the Problems and Software (including Coot) – Paul Emsley [<a href="https://youtu.be/a_fJ6kAcsa4">Video</a>]</li>
</ul>
</li>
<li>
<p><a href="https://www2.mrc-lmb.cam.ac.uk/research/scientific-training/electron-microscopy/">Electron Cryo-microscopy Course 2017, MRC-LMB</a>: Below are recordings of the LMB’s 2017 Cryo-microscopy course.</p>
<ul>
<li>Cryo-EM17 Lecture 01: Past Present Future [<a href="https://youtu.be/aHhmnxD6RCI">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-01-past-present-future/?wpdmdl=18763&amp;refresh=68defc2580e3c1759444005">Lecture Slide, PDF</a>]</li>
<li>Cryo-EM17 Lecture 02: Physics Optics [<a href="https://youtu.be/nQ-9QmRxDmA">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-02-physics-optics/?wpdmdl=18757&amp;refresh=68defc24bc6661759444004">Lecture Slide, PDF</a>]</li>
<li>Cryo-EM17 Lecture 04: Data Acquisition [<a href="https://youtu.be/9AMSabcTN24">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-04-data-acquisition/?wpdmdl=18759&amp;refresh=68defc24e19ae1759444004">Lecture Slide, PDF</a>]</li>
<li>Cryo-EM17 Lecture 06: Refinement [<a href="https://youtu.be/TfLFCeehfjM">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-06-refinement/?wpdmdl=18760&amp;refresh=68defc2514c571759444005">Lecture Slide, PDF</a>]</li>
<li>Cryo-EM17 Lecture 07: Data processing strategy - Rafael Fernandez-Leiro</li>
<li>Cryo-EM17 Lecture 08: Modelling – Alan Brown [<a href="https://youtu.be/byAFhhDt-f4">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-08-modelling/?wpdmdl=18761&amp;refresh=68defc2537ecc1759444005">Lecture Slide, PDF</a>]</li>
<li>Cryo-EM17 Lecture 09: Tomography [<a href="https://youtu.be/4Z0sQ_GhBkk">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/cryo-em17-lecture-09-tomography/?wpdmdl=18762&amp;refresh=68defc255dd601759444005">Lecture Slide, PDF</a>]</li>
</ul>
</li>
</ol>
<h4 id="service-centers">Service Centers:</h4>
<p>1.<a href="https://emcore.ucsf.edu/">UCSF EM Core, University of California San Francisco</a></p>
<ol start="2">
<li>
<p><a href="https://cryoem.wisc.edu/">The UW-Madison Cryo-Electron Microscopy Research Center (CEMRC), University of Wisconsin-Madison</a></p>
</li>
<li>
<p><a href="https://cryoem.mit.edu/">MIT.nano</a>, MIT campus</p>
</li>
</ol>
<ul>
<li>The Cryo-Electron Microscopy Facility in MIT.nano enables automated electron imaging of cryogenically preserved specimens. The facility was established in 2018 with support from The Arnold and Mabel Beckman Foundation, an Anonymous donor, as well as institutional support from the Vice President for Research, School of Science, and Departments of Biology, Materials Science, and Chemistry. The facility includes two cryoTEMs, a cryoFIB-SEM, two different cryoplungers, and a laboratory space for cryo-specimen preparation.</li>
</ul>
<ol start="4">
<li>
<p><a href="">Bio-Electron Microscopy Facility, Tsinghua University</a></p>
</li>
<li>
<p><a href="https://cryoem.sustech.edu.cn/">Bio-Electron Microscopy Facility, Southern University of Science and Technology</a></p>
</li>
</ol>
<ul>
<li>Five 300-kV Titan Krios (FEI)</li>
<li>One 120-KV Talos L120C G2</li>
</ul>
<ol start="6">
<li>
<p><a href="https://bioem.shanghaitech.edu.cn/7390/list.htm">Bio-Electron Microscopy Facility, Shanghai Tech University</a></p>
</li>
<li>
<p><a href="">IOCB, Prague</a></p>
</li>
</ol>
<ul>
<li>One 300-kV Titan Krios (FEI) (equipped with a Gatan K3 (model 1025) <strong>direct electron detector</strong> mounted on a Gatan BioQuantum (model 1967) <strong>energy filter</strong>)</li>
<li>One 200-KV</li>
<li>One 120-KV</li>
</ul>
<h4 id="vendor-companies">Vendor companies</h4>
<ol>
<li><a href="https://www.thermofisher.com/us/en/home/electron-microscopy/products/sample-preparation-equipment-em/vitrobot/instruments/vitrobot-mark-iv.html">Thermo Scientific</a></li>
<li><a href="https://shuimubio.com/about">Shuimu Bioscience</a>: Shuimu is a rapidly evolving global cryo-EM &amp; AI platform dedicated to early drug discovery. Equipped with 8 state-of-the-art 300kV cryo-EM instruments, Shuimu drives innovation to accelerate the development of groundbreaking drugs.</li>
<li><a href="https://www.leica-microsystems.com/products/sample-preparation-for-electron-microscopy/p/leica-em-gp2/app/">Leica</a></li>
<li><a href="https://cryowrite.ch/">SPT Labtech</a></li>
<li><a href="https://support.shimadzu.com.cn/an/resource/index.html#105">cryoWrite</a></li>
<li><a href="https://cryosol-world.com/vitrojet-solutions/technology/">CryoSol</a></li>
<li><a href="https://www.quantifoil.com/">Quantifoil, Germany</a>: Founded in 1999, Quantifoil Micro Tools GmbH was the first commercial manufacturer of electron microscopy support films of predefined hole size and arrangement and continues to set the global standard. Quantifoil&rsquo;s cryo-EM sample supports have been cited in more than 4,000 peer-reviewed publications, and are essential to cryo-EM workflows worldwide.</li>
</ol>
<h3 id="ref-labs">Ref labs:</h3>
<ol>
<li><a href="https://cryoem.ucsf.edu/">Yifan Cheng Lab, UCSF</a></li>
</ol>
<ul>
<li><strong>Atomic structures of TRPV1</strong>: With David Julius’s laboratory at UCSF [<a href="https://www.nature.com/articles/nature12822#Abs1">Liao, et al. 2013, Nature</a> | <a href="https://doi.org/10.1038/nature12823">Cao et al. 2013, Nature</a> | Gao et al. 2016, Nature]</li>
<li><strong>iBiology Talks</strong>: [<a href="https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EfUivsENbh5Gj8E10GGp0KEBynIVfURLM06aJ4xv6Y3XxQ?e=I8Ahs5">Part 1: Single Particle Cryo-EM</a> | <a href="https://pkueducn-my.sharepoint.com/:v:/g/personal/lijun0705_pku_edu_cn/EXRwR0JzrnBOoKHcQUQvjNMB5kaISskZMilMzXJFH2FAAw?e=05Bk8U">Part 2: Single particle Cyro-EM of membrane proteins</a>]</li>
</ul>
<ol start="2">
<li>
<p>Niko Grigorieff Lab, <a href="https://www.janelia.org/our-research/former-labs/grigorieff-lab">Janelia farm</a>/<a href="https://grigoriefflab.umassmed.edu/research_interests">UMASS</a>: The Grigorieff lab has played a foundational role in modern single-particle cryo-electron microscopy by developing widely adopted algorithms and software for motion correction, contrast transfer function estimation, and high-resolution 3D reconstruction, including CTFFIND, Unblur, Frealign, and cisTEM. These contributions have significantly advanced the resolution, robustness, and accessibility of cryo-EM, transforming it into a routine tool for structural biology. Their work exemplifies how rigorous physical modeling and computational innovation can unlock biological insight from complex and noisy imaging data.</p>
</li>
<li>
<p><a href="https://jensenlab.caltech.edu/">Grant Jensen Lab, Caltech</a>: Grant then entered an M.D./Ph.D. program at Stanford University, and earned his doctorate in Biophysics working on electron microscopy of RNA polymerase and other protein complexes with <strong>Dr. Roger Kornberg</strong> (who later won the Nobel prize for structural studies of transcription).</p>
</li>
</ol>
<ul>
<li><strong>Course</strong>: <a href="https://cryo-em-course.caltech.edu/unit-1-outline">“Getting Started in Cryo-EM”</a>
<ul>
<li>Part 1: Currents, coils, knobs and names: Basic anatomy of the electron microscope [<a href="https://cryo-em-course.caltech.edu/documents/8696/part_1-_em_anatomy_cBGbc1R.pdf">PDF</a> | <a href="https://youtu.be/GBU1eA1PqeQ?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 2: Fourier transforms and reciprocal space for beginners [<a href="https://cryo-em-course.caltech.edu/documents/8699/part_2_-_fouriertransforms_gIqtOnD.pdf">PDF</a> | <a href="https://youtu.be/-EAQm8wgLbc?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 3: Image formation [<a href="https://cryo-em-course.caltech.edu/documents/8698/part_3_-_image_formation_47kqLDj.pdf">PDF</a> | <a href="https://youtu.be/gmZ-vzB5lyI?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 4: Fundamental challenges in biological EM [<a href="https://cryo-em-course.caltech.edu/documents/8701/part_4_-_fundamentalchallenges_21NEmk3.pdf">PDF</a> | <a href="https://youtu.be/jYhp4aLZEb4?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 5: Tomography [<a href="https://cryo-em-course.caltech.edu/documents/14740/part_5_-_Review_slides_tomography_Revised.pdf">PDF</a> | <a href="https://youtu.be/tOnijBqdJ1E?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 6: Single-particle analysis [<a href="https://cryo-em-course.caltech.edu/documents/8702/part_6_-_single_particle_analysis_3bTNWgw.pdf">PDF</a> | <a href="https://youtu.be/MkzZNrrrJpc?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Part 7: 2-D crystallography [<a href="https://cryo-em-course.caltech.edu/documents/8697/part_7_-_electroncrystallography_qQ7rp3w.pdf">PDF</a> | <a href="https://youtu.be/Qn7aP-fRT5I?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-">Lecture video</a>]</li>
<li>Concept Check Questions [<a href="https://cryo-em-course.caltech.edu/documents/14853/cryo-em_concept_check_questions_Revised_1.pdf">PDF</a>]</li>
<li>Unit 2: Sample Preparation [<a href="https://www.youtube.com/playlist?list=PL8_xPU5epJdfd5fM2CjQItR-iRlIEIJk8">Hands-on videos</a>]
<ul>
<li><strong>(1) Preparing hydrophilic grids</strong>: [Manufacturer&rsquo;s grid boxes | Recognizing the carbon and copper sides of a grid | How to grab grids with tweezers | Glow discharging Plasma cleaning | Hydrophobic and hydrophilic grids]</li>
<li><strong>(2) Plunge freezing with a Vitrobot</strong>: [Plunge freezing tools | Vitrobot connections | Mounting the Vitrobot humidifier | Filling the Vitrobot humidifier | Tour of the Vitrobot humidifying chamber | The Vitrobot user interface and options | Replacing Vitrobot blot pads | Mounting and unmounting Vitrobot tweezers | Aligning the Vitrobot tweezers between the blot pads | Preparing the ethane gas source | Filling the ethane cup | Plunge freezing with the Vitrobot | Reviving the ethane | Sources of contamination]</li>
<li><strong>(3) Assembling autogrids</strong>: [Autogrid assembly tools | Loading C clips into insertion tools | room temp demo | Assembling autogrids (cryo conditions)]</li>
<li><strong>(4) Loading autogrids into cassettes</strong>: [Cassette loading tools | Assembling the cassette loading station | Structure of the cassette | Inserting autogrids into the cassette | Loading the cassette - room temp demo | Loading the cassette (cryo conditions) | ]</li>
<li><strong>(5) Docking and undocking capsules</strong>: [The capsule pin and its purpose | Docking a cassette | Undocking a cassette]</li>
<li><strong>(6) Plunge-freezing safety</strong>: [Biological hazards | Compressed gas hazards | Explosive gas hazards | Can the liquid ethane in the cup catch fire? | Large liquid nitrogen tank hazards | Safety Small liquid nitrogen dewars | Liquid nitrogen on the skin hazards | Ethane splash and cold cup hazards]</li>
</ul>
</li>
</ul>
</li>
</ul>
<ol start="4">
<li><a href="https://ygshi.org/">Yigong Shi lab, Princeton University/Tsinghua University/Westlake University</a></li>
</ol>
<ul>
<li><strong>Structural basis of pre-mRNA splicing</strong>
<ul>
<li>Yeast system</li>
</ul>
</li>
<li><strong>Structural basis of Apoptosis (Programmed Cell Death)</strong></li>
<li><strong>Alzheimer&rsquo;s Disease (AD) and Regulated Intramembrane Proteolysis (RIP)</strong></li>
</ul>
<ol start="5">
<li><a href="https://routlab.rockefeller.edu/">Michael P. Rout lab, The Rockefeller University</a></li>
</ol>
<ul>
<li><strong>Structure, Mechanism and Evolution of the Nuclear Pore Complex</strong>
<ul>
<li>Review: <a href="https://routlab.rockefeller.edu/wp-content/uploads/2023/02/23_2001_JBC_Rout_Aitchison.pdf">Rout &amp; Aitchison, 2001</a></li>
<li>Yeast system [<a href=""></a>|]</li>
</ul>
</li>
</ul>
<ol start="6">
<li><a href="https://www.wulab.tch.harvard.edu/">Hao Wu lab, Harvard Medical School</a></li>
</ol>
<ul>
<li><strong>The discoverer of signalosomes</strong>, which are large macromolecular complexes involved in cell death and in innate and adaptive immune pathways.
<ul>
<li>Wu, Hao. &ldquo;Higher-order assemblies in a new paradigm of signal transduction.&rdquo; Cell 153.2 (2013): 287-292. [<a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ES4S-RQx0FFEsL1XMYEsweQBeAjiV5qHw0yrYIDRkcZmMQ?e=8L9mMo">PDF</a> | <a href="https://doi.org/10.1016/j.cell.2013.03.013">DOI Link</a>]</li>
</ul>
</li>
</ul>
<ol start="7">
<li><a href="https://scholar.princeton.edu/yanlab">Nieng Yan lab, Tsinghua University/Princeton University/ShenZhen Medical Institute</a></li>
</ol>
<ul>
<li><strong>Structural basis of Membrane transport</strong>
<ul>
<li>human GLUT1</li>
</ul>
</li>
<li><strong>Structural basis of ion channels</strong>
<ul>
<li>Nav-1.7</li>
<li>Cav-3.1</li>
</ul>
</li>
<li><strong>Structural and mechanistic investigation of sterol homeostasis</strong></li>
</ul>
<ol start="8">
<li><a href="https://membrane.berkeley.edu/">James H. Hurley Lab, University of California Berkeley</a></li>
</ol>
<ul>
<li><strong>Structural basis of Lysosomal Regulation</strong></li>
</ul>
<ol start="9">
<li><a href="">Xiaochen Bai Lab, UTSW</a></li>
</ol>
<hr>
<h1 id="x-ray-crystallography">X-ray crystallography</h1>
<blockquote>
<p>While the X-ray evidence cannot, at present, be taken as direct proof that the structure is helical, other considerations discussed below make the existence of a helical structure highly probable.
&mdash; Rosalind Franklin</p>
</blockquote>
<hr>
<h3 id="principle-of-technology">Principle of technology</h3>
<p>Crystals are formed by the periodic arrangement of atoms, ions, or molecules in space, and this periodic lattice structure provides the fundamental basis for X-ray diffraction (XRD). X-ray diffraction technology enables the precise determination of the spatial positions of these constituents within a crystal, allowing for the inference of substance structures. Consequently, it has become one of the primary technical means for studying the structures of biological macromolecules.</p>
<p>The field of single-crystal diffraction for biological macromolecules evolved significantly after the 1950s. By the 1970s, it had matured into a specialized branch of crystallography known as Protein Crystallography.</p>
<h4 id="methodologies-in-x-ray-crystallography">Methodologies in X-ray Crystallography</h4>
<ul>
<li><strong>SIR (Single Isomorphous Replacement)</strong>: A traditional method involving the introduction of a single heavy atom into the crystal to determine phases.</li>
<li><strong>MR (Molecular Replacement)</strong>: A technique that utilizes a known homologous protein structure as a search model to determine the orientation and position of the molecule in a new crystal (<a href="https://doi.org/10.1107/S0365110X62000067">Rossmann and Blow, 1962</a>).</li>
<li><strong>MAD (Multi-wavelength Anomalous Diffraction)</strong>: A method that collects data at multiple wavelengths around the absorption edge of an anomalous scatterer (e.g., selenium or a heavy atom) to solve the phase problem (<a href="https://pubmed.ncbi.nlm.nih.gov/4079795/">Hendrickson, 1985</a>).</li>
<li><strong>SAD (Single-wavelength Anomalous Diffraction)</strong>: A streamlined approach that uses anomalous scattering data collected at a single wavelength to determine phases.</li>
<li><strong>Direct Methods</strong>: Mathematical approaches that derive phases directly from the observed intensities of the diffraction spots, typically used for small molecules or very high-resolution protein data.</li>
</ul>
<h3 id="vendor-companies-and-x-ray-diffractometers">Vendor companies and X-ray Diffractometers</h3>
<ol>
<li><a href="https://rigaku.com/products/x-ray-diffraction-and-scattering/xrd">Rigaku, Japan</a></li>
</ol>
<ul>
<li>Rigaku MicroMax-007HF</li>
</ul>
<ol start="2">
<li><a href="https://www.bruker.com/en/products-and-solutions/diffractometers-and-x-ray-microscopes/x-ray-diffractometers.html">Bruker, USA</a></li>
</ol>
<ul>
<li>The D8 DISCOVER Family</li>
<li>D8 QUEST</li>
</ul>
<ol start="3">
<li><a href="https://www.sptlabtech.com/products/mosquito/mosquito-crystal">sptlabtech, UK</a></li>
</ol>
<ul>
<li>mosquito: For high-throughput protein crystallization screenings, the <strong>Mosquito Crystal Robot</strong> serves as a premier liquid handling platform. It seamlessly facilitates both <strong>vapor diffusion</strong> and <strong>lipidic cubic phase (LCP)</strong> methodologies. The system supports standardized protocols for sitting drop and microbatch techniques across all industry-standard multi-well plates, while also enabling automated hanging drop setups in 96-well formats.</li>
</ul>
<h3 id="ref-labs-1">Ref. Labs</h3>
<h3 id="pioneers">Pioneers</h3>
<ol>
<li>Rosalind Franklin (1920-1958)</li>
</ol>
<h4 id="online-resources-1">Online resources</h4>
<ol>
<li><a href="https://www2.mrc-lmb.cam.ac.uk/research/scientific-training/crystallography-course-2013/">Crystallography Course 2013, MRC-LMB</a>
<ul>
<li>Molecular replacement [<a href="https://www2.mrc-lmb.cam.ac.uk/download/molecular-replacement/?wpdmdl=18799&amp;refresh=68df0795bcf791759446933">Lecture slides, PDF</a>]</li>
<li>03 Protein Expression Crystallization And Mutagenesis – Meindert Lamers [<a href="https://youtu.be/glQXW7MyIts">Video</a> | <a href="https://www2.mrc-lmb.cam.ac.uk/download/03-protein-expression-crystallization-and-mutagenesis-meindert-lamers/?wpdmdl=18793&amp;refresh=68df0795ddcca1759446933">Lecture slides, PDF</a>]</li>
<li>LMB Crystallography Course, 2013 [<a href="https://www2.mrc-lmb.cam.ac.uk/download/lmb-crystallography-course-2013/?wpdmdl=18792&amp;refresh=68df0796e2a431759446934">Lecture slides, PDF</a>]</li>
<li>Model-Building of Proteins Using X-ray Data With Coot Paul Emsley [<a href="https://www2.mrc-lmb.cam.ac.uk/download/model-building-of-proteins-using-x-ray-data-with-coot-paul-emsley/?wpdmdl=18785&amp;refresh=68df07978699a1759446935">Lecture slides, PDF</a>]</li>
</ul>
</li>
</ol>
<hr>
<h1 id="nmr">NMR</h1>
<hr>
<h3 id="ref-labs-2">Ref. Labs</h3>
<ol>
<li><a href="http://tanglab.cn/about.html">Chun Tang Lab</a>, Peking University</li>
</ol>
<hr>
<h1 id="structural-proteomics">Structural Proteomics</h1>
<h2 id="ref-labs-3">Ref. Labs</h2>
<ol>
<li><a href="https://msbiopolymers.group.uochb.cz/en">Mass Spectrometry of Biopolymers (BioMS), Core facility, IOCB-Prague</a></li>
</ol>
<ul>
<li>
<p><strong>HDX-MS:</strong> Hydrogen-Deuterium Exchange Mass Spectrometry (HDX-MS) is a structural proteomics method based on a chemical reaction that allows the exchange of covalently bound hydrogen atoms for deuterium from the solution. This exchange occurs under physiological conditions, and its rate depends on the local secondary structure of proteins. By performing a differential experiment under different conditions, it is possible to track the rate of exchange at the peptide level to localize structural changes on proteins due to the binding of ligand (identification of the binding site), another protein (identification of the interaction interface), or due to changes in physicochemical conditions (temperature, pH), including accompanying allosteric effects.</p>
</li>
<li>
<p><strong>Crosslinking (XL-MS):</strong> This method is based on crosslinking proteins using reagents of a specific length, which, after binding to the protein and subsequent mass spectrometric detection, provide information about the distances between two amino acid side chains. This technique can detect and structurally characterize protein-protein interactions or characterize distances within the structure of an individual protein. The output is a table listing reliable detected crosslinks and their graphical representation on the protein sequence and on a 3D model.</p>
</li>
</ul>
<h1 id="other-technologies">Other technologies</h1>
<p>smFRET，FLIM-FRET</p>
<hr>
<h1 id="structure-modeling">Structure modeling</h1>
<ul>
<li>AlphaFoldv3</li>
<li>molecular dynamics simulations
<ul>
<li>The CHARMM-GUI</li>
<li>The CHARMM36m force field</li>
<li>Gromacs 2023</li>
</ul>
</li>
</ul>
<p><strong>Ref database</strong>:</p>
<ol>
<li>EMDB: <a href="https://www.ebi.ac.uk/pdbe/emdb/">https://www.ebi.ac.uk/pdbe/emdb/</a></li>
<li>EMPIAR: <a href="https://www.ebi.ac.uk/pdbe/emdb/empiar/">https://www.ebi.ac.uk/pdbe/emdb/empiar/</a></li>
</ol>
<hr>
<ol start="3">
<li>Protein structure database</li>
</ol>
<table>
<thead>
<tr>
<th>Protein Name</th>
<th>PDB No.</th>
<th>Species</th>
<th>Expression System</th>
<th>Best Resolution (Å)</th>
<th>EM Data Bank</th>
</tr>
</thead>
<tbody>
<tr>
<td>BRAF</td>
<td><a href="https://www.rcsb.org/structure/8C7X">8C7X; 8C7Y</a></td>
<td>Homo sapiens</td>
<td>Escherichia coli</td>
<td>1.65</td>
<td></td>
</tr>
<tr>
<td>AKT1</td>
<td><a href="https://www.rcsb.org/structure/1UNQ">1UNQ</a></td>
<td>Homo sapiens</td>
<td>Escherichia coli BL21</td>
<td>0.98</td>
<td></td>
</tr>
<tr>
<td>mTOR</td>
<td><a href="https://www.rcsb.org/structure/5H64">5H64</a></td>
<td>Homo sapiens</td>
<td>HEK293F</td>
<td>4.4</td>
<td></td>
</tr>
<tr>
<td>mTORC1-TFEB-Rag-Ragulator</td>
<td><a href="https://www.rcsb.org/structure/7UXH">7UXH</a></td>
<td>Homo sapiens</td>
<td>HEK293F</td>
<td>3.4</td>
<td></td>
</tr>
<tr>
<td>Raptor-TFEB-Rag-Ragulator</td>
<td><a href="https://www.rcsb.org/structure/7UX2">7UX2</a></td>
<td>Homo sapiens</td>
<td>HEK293F</td>
<td>2.9</td>
<td></td>
</tr>
<tr>
<td>mTORC1-RAG-Ragulator-4E-BP1</td>
<td><a href="https://www.rcsb.org/structure/7UXH">7UXH</a></td>
<td>Homo sapiens</td>
<td>HEK293F</td>
<td>3.23</td>
<td><a href="https://www.ebi.ac.uk/emdb/EMD-47932?tab=experiment">EMD-47932</a></td>
</tr>
</tbody>
</table>
<hr>
<h2 id="nobel-laureates">Nobel laureates</h2>
<h2 id="nobelxqhemoximgnobel_prizejpg"><img src="https://leoj.xyz/img/nobel_prize.jpg" alt="nobelxqhemox"></h2>
<ul>
<li>
<p><strong>Jacques Dubochet</strong>, University of Lausanne, Lausanne, Switzerland<br>
The Nobel Prize in Chemistry 2017<br>
<strong>Early cryo-electron microscopy</strong> [<a href="https://www.youtube.com/watch?v=Blm3QwT-rRk&amp;ab_channel=NobelPrize">Nobel Lecture video</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EQ3Lh8WjagRFoiNVbUsIbk0BbkzwPzSe2gswYWApBQQSoA?e=70flib">Lecture Slides</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERsnFw1YSGpKmK-x6I-OcY8B_SnYvmbc9TuU_qTaM_bA4Q?e=GYkHk2">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2017/dubochet/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Joachim Frank</strong>, Columbia University, New York, NY, USA<br>
The Nobel Prize in Chemistry 2017<br>
<strong>Single-Particle Reconstruction – Story in a Sample</strong> [<a href="https://www.youtube.com/watch?v=Blm3QwT-rRk&amp;ab_channel=NobelPrize">Nobel Lecture video</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EQ3Lh8WjagRFoiNVbUsIbk0BbkzwPzSe2gswYWApBQQSoA?e=70flib">Lecture Slides</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERsnFw1YSGpKmK-x6I-OcY8B_SnYvmbc9TuU_qTaM_bA4Q?e=GYkHk2">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2017/frank/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Richard Henderson</strong>, MRC Laboratory of Molecular Biology, Cambridge, United Kingdom<br>
The Nobel Prize in Chemistry 2017<br>
<strong>From Electron Crystallography to Single Particle cryoEM</strong> [<a href="https://www.youtube.com/watch?v=Blm3QwT-rRk&amp;ab_channel=NobelPrize">Nobel Lecture video</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/EQ3Lh8WjagRFoiNVbUsIbk0BbkzwPzSe2gswYWApBQQSoA?e=70flib">Lecture Slides</a> | <a href="https://pkueducn-my.sharepoint.com/:b:/g/personal/lijun0705_pku_edu_cn/ERsnFw1YSGpKmK-x6I-OcY8B_SnYvmbc9TuU_qTaM_bA4Q?e=GYkHk2">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2017/henderson/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Thomas A. Steitz</strong>, Yale University, New Haven, CT, USA; Howard Hughes Medical Institute, USA
The Nobel Prize in Chemistry 2009<br>
<strong>From the Structure and Function of the Ribosome to New Antibiotics</strong>  [<a href="https://nobel-videocdn01.azureedge.net/video/lecture_2009_che_steitz_01_496.mp4">Nobel Lecture video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/steitz-lecture-slides.pdf">Lecture Slides</a> | <a href="https://www.nobelprize.org/uploads/2018/06/steitz_lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2009/steitz/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Venkatraman Ramakrishnan</strong>, MRC Laboratory of Molecular Biology, Cambridge, United Kingdom
The Nobel Prize in Chemistry 2009<br>
<strong>Unraveling the Structure of the Ribosome</strong> [<a href="https://nobel-videocdn01.azureedge.net/video/lecture_2009_che_ramakrishnan_01_496.mp4">Nobel Lecture video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/ramakrishnan-lecture-slides.pdf">Lecture Slides</a> | <a href="https://www.nobelprize.org/uploads/2018/06/ramakrishnan_lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2009/ramakrishnan/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Roger D. Kornberg</strong>, Stanford University, Stanford, CA, USA
The Nobel Prize in Chemistry 2006<br>
<strong>The Molecular Basis of Eukaryotic Transcription</strong>  [<a href="https://nobel-videocdn01.azureedge.net/video/lecture_2006_che_kornberg_01_496.mp4">Nobel Lecture video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/kornberg-slides.pdf">Lecture Slides</a> | <a href="https://www.nobelprize.org/uploads/2018/06/kornberg_lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2006/kornberg/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Roderick MacKinnon</strong>, Rockefeller University, New York, NY, USA; Howard Hughes Medical Institute, USA<br>
Nobel Prize in Chemistry 2003<br>
<strong>Potassium Channels and the Atomic Basis of Selective Ion Conduction</strong> [<a href="https://nobel-videocdn01.azureedge.net/video/lecture_2003_che_mckinnon_01_496.mp4">Nobel Lecture video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/mackinnon-lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2003/mackinnon/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Kurt Wüthrich</strong>, Eidgenössische Technische Hochschule (Swiss Federal Institute of Technology), Zurich, Switzerland; The Scripps Research Institute, La Jolla, CA, USA
The Nobel Prize in Physics 2002<br>
<strong>NMR Studies of Structure and Function of Biological Macromolecules</strong> [<a href="https://nobel-videocdn01.azureedge.net/video/lecture_2002_che_wuthrich_01_496.mp4">Nobel Lecture video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/wutrich-lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/chemistry/2003/mackinnon/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Linus Carl Pauling (C)</strong>,
The Nobel Prize in Physics 1954\</p>
</li>
<li>
<p><strong>James Batcheller Sumner</strong>,
The Nobel Prize in Physics 1946\</p>
</li>
<li>
<p><strong>Theodor (The) Svedberg (C)</strong>,
The Nobel Prize in Physics 1926\</p>
</li>
<li>
<p><strong>Sir William Henry Bragg</strong>,
The Nobel Prize in Physics 1915\</p>
</li>
<li>
<p><strong>Max von Laue</strong>, Frankfurt-on-the-Main University, Frankfurt-on-the-Main, Germany<br>
The Nobel Prize in Physics 1914<br>
<strong>Concerning the Detection of X-ray Interferences</strong> [<a href="https://nobel-videocdn01.azureedge.net/video/docu_1914_phy_vonlaue_01_496.mp4">Nobel Documentary video</a> | <a href="https://www.nobelprize.org/uploads/2018/06/laue-lecture.pdf">Read the Lecture</a> | <a href="https://www.nobelprize.org/prizes/physics/1914/laue/facts/">Source</a>]</p>
</li>
<li>
<p><strong>Wilhelm Conrad Röntgen</strong>, Munich University, Munich, Germany<br>
The Nobel Prize in Physics 1901<br>
<strong>in recognition of the extraordinary services he has rendered by the discovery of the remarkable rays subsequently named after him</strong></p>
</li>
</ul>
              


            </div>
          </div>
          <div id="post-footer" class="post-footer main-content-wrap">
            
              
                
                
                  <div class="post-footer-tags">
                    <span class="text-color-light text-small">TAGGED IN</span><br/>
                    
  <a class="tag tag--primary tag--small" href="https://leoj.xyz/tags/cryoem/">CryoEM</a>

  <a class="tag tag--primary tag--small" href="https://leoj.xyz/tags/x-ray/">X-Ray</a>

  <a class="tag tag--primary tag--small" href="https://leoj.xyz/tags/protein-purification/">Protein purification</a>

                  </div>
                
              
            
            

            
  
    <div id="disqus_thread">
      <noscript>Please enable JavaScript to view the comments powered by Disqus.</noscript>
    </div>
    <script type="text/javascript">
      var disqus_config = function() {
        this.page.url = 'https:\/\/leoj.xyz\/resource\/structure_bio\/';
        
          this.page.identifier = '\/resource\/structure_bio\/'
        
      };
      (function() {
        
        
        if (["localhost", "127.0.0.1"].indexOf(window.location.hostname) != -1) {
          document.getElementById('disqus_thread').innerHTML = 'Disqus comments not available by default when the website is previewed locally.';
          return;
        }
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        var disqus_shortname = 'hugo-tranquilpeak-theme';
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
      })();
    </script>
  


          </div>
        </article>
        <footer id="footer" class="main-content-wrap">
  <span class="copyrights">
    &copy; 2026 Jun-Liszt Li. All Rights Reserved
  </span>
</footer>

      </div>
      <div id="bottom-bar" class="post-bottom-bar" data-behavior="4">
        

      </div>
      
<div id="share-options-bar" class="share-options-bar" data-behavior="4">
  <i id="btn-close-shareoptions" class="fa fa-times"></i>
  <ul class="share-options">
    
      <li class="share-option">
        <a class="share-option-btn" target="new" href="https://twitter.com/intent/tweet?text=https%3A%2F%2Fleoj.xyz%2Fresource%2Fstructure_bio%2F" aria-label="Share on Twitter">
          <i class="fab fa-twitter" aria-hidden="true"></i><span>Share on Twitter</span>
        </a>
      </li>
    
      <li class="share-option">
        <a class="share-option-btn" target="new" href="https://www.linkedin.com/sharing/share-offsite/?url=https%3A%2F%2Fleoj.xyz%2Fresource%2Fstructure_bio%2F" aria-label="Share on Linkedin">
          <i class="fab fa-linkedin" aria-hidden="true"></i><span>Share on Linkedin</span>
        </a>
      </li>
    
  </ul>
</div>
<div id="share-options-mask" class="share-options-mask"></div>


    </div>
    
    <div id="about">
  <div id="about-card">
    <div id="about-btn-close">
      <i class="fa fa-times"></i>
    </div>
    
      <img id="about-card-picture" src="https://leoj.xyz/img/Dingtalk_20220815163324.jpg" alt="Author&#39;s picture" />
    
    <h4 id="about-card-name">Jun-Liszt Li</h4>
    
      <div id="about-card-bio">Hello, My name is Jun-Liszt Li (Leo). my scientific interest is to understand the molecular logic of cell growth and develop new tools to address specific questions in this path.</div>
    
    
      <div id="about-card-job">
        <i class="fa fa-briefcase"></i>
        <br/>
        Postdoc fellow
      </div>
    
    
      <div id="about-card-location">
        <i class="fa fa-map-marker-alt"></i>
        <br/>
        Prague, Czech Republic
      </div>
    
  </div>
</div>

    

    
  
    
      <div id="cover" style="background-image:url('https://leoj.xyz/img/cover2.jpg');"></div>
    
  


    
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js" integrity="sha512-894YE6QWD5I59HgZOGReFYm4dnWc1Qt5NtvYSaNcOP+u1T9qYdvdihz0PPSiiqn/+/3e7Jo4EaG7TubfWGUrMQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/highlight.min.js" integrity="sha512-z+/WWfyD5tccCukM4VvONpEtLmbAm5LDu7eKiyMQJ9m7OfPEDL7gENyDRL3Yfe8XAuGsS2fS4xSMnl6d30kqGQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

<script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha512-uURl+ZXMBrF4AwGaWmEetzrd+J5/8NRkWAvJx5sbPSSuOb0bZLqf+tOzniObO00BjHa/dD7gub9oCGMLPQHtQA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>


<script src="https://leoj.xyz/js/script-yqzy9wdlzix4lbbwdnzvwx3egsne77earqmn73v9uno8aupuph8wfguccut.min.js"></script>


  
    <script async crossorigin="anonymous" defer integrity="sha512-gE8KAQyFIzV1C9+GZ8TKJHZS2s+n7EjNtC+IMRn1l5+WYJTHOODUM6JSjZhFhqXmc7bG8Av6XXpckA4tYhflnw==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/apache.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-EWROca+bote+7Oaaar1F6y74iZj1r1F9rm/ly7o+/FwJopbBaWtsFDmaKoZDd3QiGU2pGacBirHJNivmGLYrow==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/go.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-GDVzAn0wpx1yVtQsRWmFc6PhJiLBPdUic+h4GWgljBh904O3JU10fk9EKNpVyIoPqkFn54rgL2QBG4BmUTMpiQ==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/http.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-UgZlma8NzkrDb/NWgmLIcTrH7i/CSnLLDRFqCSNF5NGPpjKmzyM25qcoXGOup8+cDakKyaiTDd7N4dyH4YT+IA==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/less.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-lot9koe73sfXIrUvIPM/UEhuMciN56RPyBdOyZgfO53P2lkWyyXN7J+njcxIIBRV+nVDQeiWtiXg+bLAJZDTfg==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/nginx.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-Zd3e7XxHP00TD0Imr0PIfeM0fl0v95kMWuhyAS3Wn1UTSXTkz0OhtRgBAr4JlmADRgiXr4x7lpeUdqaGN8xIog==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/puppet.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-qtqDO052iXMSP+5d/aE/jMtL9vIIGvONgTJziC2K/ZIB1yEGa55WVxGE9/08rSQ62EoDifS9SWVGZ7ihSLhzMA==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/scss.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-1NmkjnEDnwwwcu28KoQF8vs3oaPFokQHbmbtwGhFfeDsQZtVFI8zW2aE9O8yMYdpdyKV/5blE4pSWw4Z/Sv97w==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/stylus.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-B2wSfruPjr8EJL6IIzQr1eAuDwrsfIfccNf/LCEdxELCgC/S/ZMt/Uvk80aD79m7IqOqW+Sw8nbkvha20yZpzg==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/swift.min.js"></script>
  

  
    <script async crossorigin="anonymous" defer integrity="sha512-28oDiQZGKUVN6wQ7PSLPNipOcmkCALXKwOi7bnkyFf8QiMZQxG9EQoy/iiNx6Zxj2cG2SbVa4dXKigQhu7GiFw==" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/languages/yaml.min.js"></script>
  


<script>
$(document).ready(function() {
  hljs.configure({ classPrefix: '', useBR: false });
  $('pre.code-highlight > code, pre > code').each(function(i, block) {
    if (!$(this).hasClass('codeblock')) {
      $(this).addClass('codeblock');
    }
    hljs.highlightBlock(block);
  });
});
</script>




    
  </body>
</html>

